---
title: manuscript
author: parris t humphrey
---

## Antibiosis affects competitive hierarchies and shapes the evolution of bacterial life history traits in a microbiome

#### Parris T. Humphrey<sup>1,2</sup>, Trang N. Nguyen<sup>3</sup>, Noah K. Whiteman<sup>2</sup>

<sup>1</sup>Organismic & Evolutionary Biology, Harvard University, Cambridge, MA 02138, USA<br>
<sup>2</sup>Integrative Biology, University of California, Berkeley, CA 94720, USA<br>
<sup>3</sup>Department of Plant Sciences, University of Arizona, Tucson, AZ 85721, USA

Corresponding author: PTH ([phumphrey@g.harvard.edu](phumphrey@g.harvard.edu))

<hr>

### Abstract

Enormous effort is being placed in developing stragegies to modify microbiome communities for therapeutic interventions in humans and growth enhancement in crops. Informing these approaches requires a detailed understanding of the nature of ecological interactions among bacteria within a microbiome community. Yet, from an life-history perspective, the traits that underlie ecological outcomes the correlations among them, and their distributions in microbiomes are poorly understood, in part because of difficulties in culturing many of these species. Thus, most studies aiming to understand the ecological forces structuring microbiome communities rely on correlational associations rather than causal ones. In this study, we help address this gap by experimentally characterizing the hierarchy of ecological interations in a relatively simple, native, endophytic bacterial community of a native, sub-alpine herbaceous plant. Specifically, we used *in vitro* assays to identify the types and strengths of ecological interactions that occur among a suite ($n=40$) of *Pseudomonas* spp. bacterial strains isolated from bittercress (Brassicaceae; *Cardamine cordifolia* A. Gray) at a single field site. Outcomes from pairwise competitions in spatial microcosms revealed strong competitive dominance of *P. fluorescens* strains over *P. syringae* strains. *P. fluorescens* strains were better resources competitors and more often produced antibiotics to which few *P. syringae* strains expressed resistance. Within each bacterial clade, growth rates of individual strains were not correlated with overall competitive ability or saturation density *in vitro*. However, *P. fluorescens* strains with shorter lag times *in vitro* were better competitors, suggesting that spatial preemption of a shared resource promoted competitive fitness. We then quantified how patterns of resistance among *P. syringae* to metabolic products produced by *P. fluorescens*  that acted as antibiotics might influence relative competitive ranks *within* the *P. syrinage* clade. We found a high proportion of competitive outcomes among *P. syringae* strains were reversed in the presence of antibiotics produced by *P. fluorescens*. These indirect benefits accrued largely to strains of lower average competitiveness. Thus, *P. fluorescens* are not only potent direct competitors of *P. syringae* strains but can also indirectly change the outcome of a large number of competitive interactions among them when they co-occur. Testing the relevance of these interaction types in plant leaves will help reveal the role of *P. fluorescens* in suppressing growth and upending the fitness ranks of phytopathogenic *P. syringae*. Our species interaction screen also has the potential to identify new antibiotics for biomedicine that vary in specificity.

**Keywords:** interference competition; species interactions; *Pseudomonas*; co-existence; facilitation; microbiome

**Author contributions:** PTH and NKW designed the study; PTH and TNN collected the data; PTH and NKW wrote the manuscript.

<hr>

### Introduction

https://msystems.asm.org/content/4/4/e00185-19
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6011841/
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6935513/
https://www.ncbi.nlm.nih.gov/pubmed/26542567

Microbiomes play a central role in mediating development, metabolism and immunity of most multicellular organisms. Although progress is being made, the ecological forces shaping the mammalian gut microbiome are notoriously difficult to study directly, given the diversity and high proportion of uncultivable members in this community. Plants are similarly affected in by their microbiomes, yet, in contrast, they possess a highly cultivable microbiome community. Microbiomes of plants therefore have potential to serve as models for understanding the ecology and evoltuion of microbiomes more generally. Moreover, there is major interest in developing plant growth-promiting bacterial (PGPB) formulations for use in agriculture, and this requires a precise undestanding of how ecological interacitons bewteen bacterial species affects host phenotypes. 

Although amensalism, commensalism and mutualism are likely to be important in shaping the structure of microbome communities, competition may be the principle ecological force (pubmed.ncbi.nlm.nih.gov/28770836/). Ecologists have long recognized that distinct forms of competition can operate simultaneously within communities: competition to exploit shared resources, and competition to interfere with another species’ ability to do so. In addition to structuring microbiome communities, competition of both types is also a potent source of natural selection (Hibbing et al. 2009; Cornforth and Foster 2013; Mitri and Richard Foster 2013). Teasing apart how exploitative and interference competition interact in a community context remains a long-standing challenge in ecology (DeLong and Vasseur 2013). In this study, we explore how species interactions influences the outcome of competitive interactions within a microbiome community, affecting the evolution of phenotypic traits (Ohgushi et al. 2012). 

Bacterial communities in nature, whether host-associated or not, typically exhibit high species diversity. Thus, the the outcome of any focal interaction may be reversed by nearby species (Rojas-Echenique and Allesina 2011). This effect is captured by the concept of intransitivity, which is the non-hierarchical arrangement of competitive abilities among diverse genotypes or species. The net effect of intransitivity is that competitiveness is highly context-dependent: one genotype’s ability to dominate another depends on the frequency of any number of other genotypes. Empirical intransitive interaction networks have been discovered both within (Sinervo and Lively 1996) and between species (Lankau and Strauss 2007), and theoretical studies have shown that introducing even modest intransitivity into competitive interaction networks can provide a buffer against species (or genotype) extinction (Laird and Schamp 2006; Laird and Schamp 2008; Rojas-Echenique and Allesina 2011; Laird 2014).


Despite extensive theoretical predictions, very little empirical work has attempted to describe how exploitative and interference competition traits interact in natural bacterial communities. Micorbiome communities are attractive for such studies because they are bounded spatially and are of practical import given their role in shaping host health. Much of the work on intransitivity in bacterial assemblages involves metabolite-mediated interference competition in pair-wise and three-way comparisons (Kerr et al. 2002; Lenski and Riley 2002; Kirkup and Riley 2004) using model strains of bacteria brought together in the laboratory. While crucial for advancing the field from a theoretical standpoint, studies are needed that directly measure the frequency of different exploitative and interference competitive abilities in natural bacterial assemblages. Studies like ours fill this gap and help provide a detailed view of the potential for facilitation and other forms of intransitivity to maintain life history diversity within natural bacterial communities.

<!-- Perhaps the most famous type of intransitivity takes the form of the rock–paper–scissors (R–P–S) game, where A invades B, B invades C, while C invades A. However, other forms of intransitivity such as facilitation may be common in nature (Brooker et al. 2008; Bronstein 2009; McIntire and Fajardo 2013).-->

<!--One avenue of facilitation within microbiome communities is through differential sensitivities of the two focal competing species to interference competition (e.g., a secreted toxin) from a third player. This scenario lies at the intersection of exploitative and interference competition, and its occurrence is likely maximized by intermediate frequencies of both the production of, and resistance, to secreted toxins.-->

The principal goals of this study were to (1) characterize the distribution of life history trait variances and co-variances among a diverse set of isolates from a wild microbiome meta-community, (2) examine how this life history variation relates to a network of competitive interactions, and (3) estimate how frequently intransitive competitive asymmetries among strains might promote co-existence. To address these goals, we build upon prior work using a natural community of endophytic *Pseudomonas* spp. bacteria isolated from a systematic survey of leaves of a perennial forb, bittercress (Brassicaceae; *Cardamine cordifolia* A. Gray) in the Rocky Mountains of North America. Bacterial strains were isolated primarily from the internal leaf (endophytic) compartment of bittercress and include a broad range of strains from the cosmopolitan phytopathogen *P. syringae* (Morris et al. 2008) and  *P. fluorescens*, a well-known member of many rhizosphere communities that was found here to be a common endophytic species. Competitive interactions between functionally diverse *P. syringae* isolates alters the success of leaf colonization by other strains, but the mechanisms are often unknown (Kinkel and Lindow 1993; Wilson and Lindow 1994a; Wilson and Lindow 1994b). 

Although the types of traits that contribute to these competitive interaction outcomes are unknown, both phenotype-modifying compounds and toxins are secreted by phyllosphere-dwelling bacteria, especially among *Pseudomonas* spp. (Lindow and Brandl 2003; Quiñones et al. 2005; Dulla and Lindow 2009; Dulla et al. 2010; Ma et al. 2013). Phyllosphere bacteria co-localize the surface and interior of leaves (Monier and Lindow 2005). Thus, there is potential for direct interactions between competing bacteria to affect the patterns of assembly within phyllosphere communities.
> []I dont understand the difference bewteen "phenotype-modifying" and "toxin" seems like metabolite that influences another species is what you mean.

We obtained 39 phyllosphere *Pseudomonas* spp. isolates from a bittercress population and a well-characterized laboratory reference strain (*P. syringae* pv. maculicola str. ES4326) widely used to infect the confamilial *Arabidopsis thaliana*. We measured the ecological outcome of pairwise competitive interactions among strains in this set, wherein strains competed for shared resources in spatially explicit microcosms. We quantified each strain’s ability to invade and defend against invasion and derived a composite measure of competitiveness that incorporated both invasive and defensive ability. We simultaneously measured each strain’s capacity to inhibit surrounding competitors and its ability to resist strains through inhibitory metabolites, including toxins. Using independent measurements of maximum rate of increase, lag phase, and maximum yield in vitro, we then determined the underlying correlates of both exploitative and interference competitive abilities, as well as effect of phylogeny on these correlations.


### Materials & Methods

#### Bacterial strains

Of the 51 *Pseudomonas* spp. strains isolated from bittercress and described by Humphrey et al. (2014), we selected a set of 40 (26 *P. syringae*, 14 *P. fluorescens*) that represent the relative phylogenetic diversity present in this community. The laboratory strain *P. syringae* pv. maculicola str. ES4326 (hereafter Psm4326) was used as a reference owing to its phylogenetic similarity to strains isolated from bittercress and its extensive characterization in the laboratory as a pathogen of *Arabidopsis thaliana* (Cui:2005dn, Cui:2002gp, Groen:2013bt, Groen:2015bv). All bacterial strains used in this study had undergone only one prior growth cycle after freezing following initial isolation on King's B plates from surface-sterilized homogenates of bittercress leaf samples (Humphrey et al 2014).

#### Individual *in vitro* growth characterization

##### Growth conditions

We characterized canonical growth traits of each bacterial strain separately by tracking cell density through a complete lag--exponential--saturation cycle *in vitro*. We re-streaked each strain from -80°C stocks onto King's B (KB) agar plates +10 mM MgSO<sup>4</sup> and incubated at 28°C for 3 days. We picked and inoculated single colonies into 1 mL minimal media at pH 5.6 ('MM'; 10 mM fructose, 10 mM mannitol, 50 mM KPO<sub>4</sub>, 7.6 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 1.7 mM MgCl<sub>2</sub>; Mudgett:1999tu, Barrett:2011fo) and grown overnight in a shaking incubator (250 rpm) at 28°C. MM at pH 5.6 has been shown to induce the expression of the type-III secretion system (T3SS) in a diversity of *Pseudomonas* spp. (Huynh:1989ux), in contrast to KB, which results in negligible Type-III Secretion System (T3SS) expression. T3SS expression was important for maximizing the potential relevance of our in vitro assay environments to those of plants, in which T3SS expression is expected. Thus, MM was used for this and all other culture experiments in this study.

Overnight cultures of each strain in MM were spun down for 3 m at $3000 \times \text{g}$ and the supernatant was replaced with 500 µL fresh MM. The density of each culture was adjusted to $\text{OD}_{600} = 0.2$ prior to 1:100 dilution into a total of 180 $\mu\text{L}$ MM inside the wells of sterile polystyrene 96-well plates (Falcon). Each 96-well plate was covered with optically clear, gas-permeable plastic tape (Sigma \#Z380059) and incubated for 60 hr in a BioTek 600 plate reader in which $\text{OD}_{600}$ measurements were taken every 5 min with continuous random orbital shaking. Identical growth assays were performed on separate days in duplicate.

##### Estimating traits related to growth

We used R package `grofit` (Kahm:2010vv) to fit smoothed functions to the bacterial growth data. Curve fits generated using logistic, Richards, Gompertz, or modified Gompertz equations failed to produce estimates with $r \geq 0.5$ and we therefore used a non-parametric locally-weighted smoothing function to estimate the following growth curve parameters: maximum growth rate $r_m$, lag phase $l$, and maximum yield $K$. Lag phase represents the length of time (min) prior to initiation of exponential growth, while maximum yield is the maximum OD<sub>600</sub> attained during 60 h of growth. Curves for long lag-phased strains never leveled off (Fig. S1, e.g. strain 17A); in these cases, $K$ was set as the final OD<sub>600</sub>. When growth trajectories exhibited multiple exponential phases (diauxic shifts), $r_m$ was estimated during the initial exponential phase (e.g. strain 20A; Fig. S1).

#### Pairwise competition assays

We conducted pairwise high-density competition assays in spatial microcosms in which a "resident" strain inoculated onto the surface of each plate competed with each "invader" strain spotted on top. In 100 mm diameter Petri dishes, we overlaid 4 mL of 0.5% (w/v) soft agar inoculated with a resident strain overtop a sterile base layer of growth medium (MM + 1% w/v agar). The top agar overlay for each plate contained a suspension of a single resident strain inoculated at $5 \times 10^{5} \text{ml}^{-1}$ while soft agar was still molten ~42°C. Once cooled, suspensions of each of the 40 invader strains were then spotted at the same concentration in 4 µL aliquots spaced every 0.5 cm in parallel rows using an 8-channel pipettor. Resident and invader strain suspensions were made from exponential phase cultures in MM 3 mL with shaking at 28°C. Plates were incubated at 28°C face up for 12 h and then face down incubation for an addition 10 days. Megacolony spots were scored by hand for growth after days 1, 3, 5, 8, and 10. Data used for the following analyses are from day 10, by which time all interactions dynamics had leveled off.

We scored growth of each invader as $0$ for no visible growth of the invader above a negative control spot containing MM alone, $0.5$ for a largely translucent 'megacolony', which reflected a definite presence of growth but which was relatively suppressed and confined to the megacolony margin, and $1$ for obvious and robust megacolony growth. Examples of each can be seen in Fig. S2. We scored inhibition interactions as the presence of a zone of clearance (halo) $\geq 1 \text{ mm}$ surrounding the extent of the invader megacolony (Fig. S3). Inhibition interactions were ultimately scored as 0 or 1 regardless of the spatial extent of the halo, although variation in halo width was recorded. We also scored any morphological variation among megacolonies for particular strains, and later we relate such variation for particular strains to the phylogenetic position of their competitors (see *Analyzing the distribution of competitive outcomes*, below).

#### Calculating indexes of competitiveness

Each strain was assayed under 40 different conditions both as resident strain and as invader, comprising an interaction network with 1600 entries (including self vs. self). One version of the interaction network represents the outcome of resource competition and details the extent of growth of each invader, while the other captures the presence or absence of inhibitory interactions indicated by zones of clearance in the resident population. For resource competitions, we calculate the invasiveness ($C_o$) and defense capacity (i.e. territoriality; $C_d$) of each strain. $C_o$ for each strain $i$ was calculated as

$$
C_{o,i} = \frac{1}{n_{ij}}\sum_{i \neq j}^{n}x_{ij}
$$

where $x_{ij} \in \{ 0,0.5,1\}$ and $n_{ij}$ is the total number of scored interactions for each strain as the invader with all non-self resident strains. $C_o$ is thus the expected value of growth attained by each strain as the invader across the population of residents.

$C_d$ quantifies the ability of each strain to resist invasion by other strains and is calculated as

$$
C_{d,j} = \frac{1}{n_{ji}}\sum_{j \neq i}^{n}(1 - x_{ji})
$$

In this equation, strain $j$ is in the resident state, and $x_{ji} \in \{ 0,0.5,1\}$ as before but with a subscript reversal, indicating the degree to which the resident prevented the growth of each invader $i$. As above, $n_{ji}$ is the number of interactions occurring between each focal resident and its non-self invaders. $C_d$ can thus be interpreted as the expected amount of growth each resident strain can prevent among the population of invaders assayed.

We then calculated an overall exploitative competition index, $C_w$, for each strain as 

$$
C_w = C_{o} - {(1 - C}_{d})
$$

where $-1 \leq C_{w} \leq 1$. These extremes represent absolute competitive inferiority ($-1$), where a strain failed to prevent any growth of any invader and similarly failed to invade any other strain, to absolute competitive dominance ($1$), where a strain fully invaded all residents and fully prevented growth of all invaders.

We also calculated $C_t$ and $C_r$ based on the interaction matrix for interference competition. Here, $C_{t}$ is the proportion of successful invasions (i.e. given growth of 0.5 or above) that also resulted in halo formation (produced by invading strain), indicative of inhibition of the resident. $C_r$ for a strain is the proportion of contests with all invading inhibitor strains (i.e., all strains with $C_t > 0$) that failed to result in halo formation, which we took as evidence of resistance. Anlogous to $C_w$ above, an overall interference competition index, $I_w$, was calculated for each strain as 

$$
I_{w} = C_{t} - {(1 - C}_{r})
$$

where $-1 \leq I_w \leq 1$, which is equal to the aggressiveness index ($AI$) of Vestigian et al. (2011).

#### Examining trait correlations

To examine fundamental axes of trait co-variances, we conducted principal components analysis (PCA) using the matrix of mean-centered and scaled competitive indexes and growth parameters for all strains (40 x 9 matrix) as input. We also constructed linear multiple regression models to estimate the contribution of $r_m$, $L$, and $K$ to variation among *P. syringae* and *P. fluorescens* strains in each of the overall competitive indexes $C_w$ and $I_w$.

#### Analyzing the distribution of competitive outcomes

We determined when outcomes of all pairwise interactions between strains $i$ and $j$ ($i \neq j$) took following forms: reciprocal invasibility (RI), where strains $i$ and $j$ each invade one another; reciprocal non-invasibility (RNI), where strains $i$ and $j$ cannot invade each other; and asymmetric (AS), where strain $i$ invades strain $j$ but $j$ cannot invade $i$. We constructed binomial generalized linear models (GLMs) in \texttt{R} with the canonical logit link function to estimate the probability of RI, RNI, and AS as a function of genetic distance as well as trait distance between strains. Genetic distance ($D_g$) was calculated as the pairwise uncorrected nucleotide distance between 2690 bp of sequence comprised of four partial housekeeping gene sequences previously generated for each strain from Humphrey et al. (2014). Orthologous sequences from the genome of Psm4326 were derived from its published genome sequence (Baltrus et al 2011; RefSeq ID `NZ_AEAK00000000.1`).

Euclidean distances between each growth trait $r_m$, $L$, and $K$ for all pairs of strains were measured as $D_{ij} = \sqrt{{(x_{i} - x_{j})}^{2}}$. We first examined a binomial model for each outcome type using $D_g$ as the only fixed effect, and then computed models including each growth trait, which took the form

$$
\text{logit}\left(P(y_{\text{ij}}|x_{\text{ij}}) \right)\ \sim\ \beta_{0} + \beta_{d}x_{d} + \beta_{r_{m}}x_{r_{m}} + \beta_{L}x_{L} + \beta_{K}x_{K}
$$

To test for genetic correlations in trait values, we ran Mantel tests between pairs of trait and genetic distance matrixes in R using package `vegan`. Test statistics were compared with those generated from 1000 matrix permutations (veganCommunityEco:2012uw). We noted instances where megacolony morphology differed between strain pairings for particular isolates (e.g., *P. fluorescens* str. RM43A), and we compared the incidence of each discrete phenotype to the phylogenetic position of the competitor strains using the genetic data described above (data from Humphrey:2014ga). To test for a correlation between the phylogenetic position of competitor strains and the induced megacolony morphology of the focal strains, we conducted a permutation analysis of variance (perMANOVA) using `vegan` package with 10000 permutations.

#### Inferring indirect interactions from the pair-wise network

We assembled all possible combinations of strain trios and evaluated whether their patterns of interactions fulfilled the criteria for facilitation described above. Specifically, we calculated the net first order linkage effects on each strain serving as the focal strain in the presence of each other as the "competitor" strain, where the interaction between the two is mediated by a nearby third strain. Briefly, facilitation can occur by strain $A$ releasing strain $C$ from inhibition from $B$ (where $A$ also has to be resistant to $B$'s inhibitors), or from resource competition from the superior competitor $B$. This analysis is agnostic to mechanism, but calculates the proportion of conditions under which facilitation of an otherwise less-fit competitor is expected to arise. We also determined the prevalence of strain trios resulting in R--P--S intransitivity and synergistic inhibition, where strain $C$ is both out-competed by $B$ and inhibited by $A$, to which the $B$ strain is resistant. For each strain, we calculated the net effect of antagonistic vs. facilitative indirect interactions across all possible trios and compared this to underlying fitness metrics derived from the pair-wise interaction network.

We calculated the magnitude of such opportunity costs of being non-resistant to quantify the predicted dependence of the indirect cost of susceptibility and the indirect benefit of resistance. We compared these two variables to their underlying dependence on relative competitiveness in terms of resource use ($C_w$). Strong competitors lose more by being sensitive, because they would have already won most resource contests. In contrast, weaker resource competitors have much to gain by being resistant, but little to lose: the relative improvement in fitness increases dramatically as more contests are won owing to their increased resistance. We modeled how the costs and benefits of susceptibility and resistance depend on underlying resource competitiveness by simulating so-called first-order linkage indirect effects, where the focal interaction is impacted by a third associate strain.


### Results

#### Exploitative competition

Pairwise soft-agar invasion assays revealed that the competitive abilities of *P. fluorescens* isolates was superior to *P. syringae* isolates across all three metrics (Fig. 1, Fig. 2). Within *P. syringae*, variation between sub-clades was evident (Fig. 1a), reflected in a moderate correlation between DG and overall competitive fitness, $C_{w}$ ($95\% < m < 99\%$ null quantiles; Table S1). Variation in competitive indexes in *P. fluorescens* was slightly bimodal (Fig. 1b): distances between strains exhibited a strong positive correlation with genetic distance ($m > 99\%$ null quantile; Table S1).

#### Interference competition

Of the 40 strains assayed, 13 (all *P. fluorescens*) produced halos surrounding some subset of the resident strains they invaded, indicating the production of diffusible inhibitors (Fig. 1; Fig. S2). Although we scored inhibition as a binary trait (see Methods), the strength and extent of inhibition varied among strain pairings, examples of which are shown in Fig. S3a–c (black arrows). Mean inhibition index ($C_{t}$) was 0.20, although two strains inhibited only one other, and 03A failed to inhibit any strains (Fig. S2). Inhibition interactions were predominately restricted to *P. fluorescens* pairings with *P. syringae*, although four *P. fluorescens* strains were susceptible to inhibition by two of the toxic strains (43A, 34A; Fig. S2). Resistance to toxin producers in *P. syringae* was variable, although the mean value was high at 0.72 (Fig. 1b). Genetic distance between strains was correlated neither with similarity in resistance values ($C_{r}$) in *P. syringae* to toxin-producers, nor with $C_{t}$ in *P. fluorescens* ($m < 90\%$ quantiles; data not shown). Overall, only *P. fluorescens* exhibited toxin production, but resistance to such toxins was common among strains from both clades.

<!-- ******** -->
<!-- FIGURE 1 -->
<!-- ******** -->
<p>
<center><img src="https://i.imgur.com/WJ4qxbP.png" width = "600"></center><br>

><strong>Fig. 1.</strong> <strong>Pairwise competitive interactions in a phyllosphere <i>Pseudomonas</i> spp. community.</strong> Rows reflect strains in the resident state, while columns reflect strains in the invader state. Dashed red lines through interaction matrix denote within–between clade divisions for ease of visualization. Phylogeny modified from Humphrey et al. (2014).
</p>

#### Life history correlates of competitiveness.

Offense ($C_{o}$) and defense ($C_{d}$) were strongly positively correlated overall with linear slopes near 1 (Fig. 2a; Fig. S4c). However, there is a dense and narrow band of strains with Co between 0.2 and 0.25 that showed a wide range of Cd values between 0.03 and 0.6 (Fig. S4c). Within this range, offense and defense appear uncorrelated, which may indicate a saturating function may fit the data better. However, a second-order polynomial regression did not improve fit (log likelihood ratio test; $\chi^{2} = 0.8$, $p > 0.2$, data not shown). Both offense and defense compared against the composite measure Cw, yielded similar near-1 slopes but with slightly offset elevations between the clades (Fig. S4c).
> [do you mean intercepts?](/FktCqC1QQkmDSaKk-2mjDQ)
elevations
<!-- ******** -->
<!-- FIGURE 2 -->
<!-- ******** -->

<!--<center><img src="https://i.imgur.com/HtPIGmq.png" width = "600"></center><br>-->
<center><img src="https://i.imgur.com/aisfE3d.png" width = "800"></center><br>

><strong>Fig. 2. Phylogenetic distribution of life history trait variation within a *Pseudomonas* spp. community.</strong>     <strong>a</strong>. Life history components are maximum growth rate ($r_{m}$), lag phase ($l$), maximum yield ($K$), derived from individual microcosm growth experiments; and components of offensive ($C_{o}$), defensive ($C_{d}$), overall ($C_{w}$) competitiveness, resistance to toxicity ($C_{r}$), and toxicity ($C_{t}$), measured on derived from a pairwise competitive interaction network (see Methods). Column $z$-score of each trait value indicated by color. <strong>b</strong>. Smoothed frequency distributions of trait values for each measured trait by clade (*P. fluorescens* and *P. syringae*). Mean ($\mu$) estimates per clade with $\pm 2\times$ standard errors depicted to the right of the curves. Note the $x$-axis value scale modifiers to the right of the axis labels. (**c**) Pairwise correlations and principle component analysis (PCA) (**d**) of six traits reflect dissimilarities between clades, as well reveal the correlational structure among traits across *Pseudomonas* spp. Correlations with text values reflect magnitude of each Pearson’s $r$ where the FDR corrected $p < 0.05$; comparisons with FDR-corrected $p<0.10$ are italicized. **d**. PCA 95% envelopes per clade depicted as solid or dashed ellipses. Dots are labeled with strain IDs. Individual trait vector loadings are in blue for resource use traits and orange for interference traits)

Competition and growth traits revealed opposing correlational patterns between the clades: $C_{w}$ was positively correlated with rm for *P. syringae* (model $F_{3,21} = 11.5$, adjusted $r^{2} = 0.57$; Table 1), but negatively with both rm and L for *P. fluorescens*. When analyzed in a multiple regression, the correlation with rm disappeared, revealing a strong negative relationship with $l$ as the sole driver of variation in $C_{w}$ for *P. fluorescens* (model $F_{3,21} = 15.3$, adjusted $r^{2} = 0.77$; Table 1). $C_{w}$ in *P. syringae*, on the other hand, was solely explained by $r_{m}$; Table 1).


PCA of $C_{w}$, $C_{r}$, and $C_{t}$ with the three growth traits revealed largely non-overlapping 95% confidence ellipses for the two clades (Fig. 2b). The first two principal components together explained 72.5% of the variation in the data. The loading vectors of competitiveness and lag phase were in opposing directions, indicating a negative correlation, while those for competitiveness and inhibitory capacity are largely colinear, indicating a positive correlation (Fig. 2b). The loading for resistance, $C_{r}$, falls close to co-linear with lag phase duration, a relationship that was not apparent in the pair-wise correlation analysis depicted in Fig. 2a. Interestingly, strain 08B—tentatively categorized as *P. syringae* in this analysis but phylogenetically sister to that clade falls beyond the 95% confidence ellipses for both named clades (Fig. 2b).

#### Genetic distance, trait matching, and competition outcomes

Asymmetric competitiveness was the most frequent interaction outcome both for the total strain set and the within- and between-clade comparisons (Fig. S6a). 98% of between-clade pairings resulted in the asymmetric dominance of one strain (the *P. fluorescens* strain always won; Fig. S2). Within clades, however, *P. syringae* strains were more likely to exhibit reciprocal invasibility than *P. fluorescens*, while the proportion of asymmetric interactions was similar in both clades (20%; Fig. S6a). RI and RNI interactions became less frequent with increasing genetic distance (Fig. S6b; Table 2). Including pairings between very same strain (i.e. the most “intra-specific” form of competition assayed here) into this GLM increased the magnitude of the probability estimate for reciprocal competitive exclusion at $D_{G} \sim 0$ (Table 2).

<!-- ******** -->
<!-- FIGURE 4 -->
<!-- ******** -->

<center>
<img src="https://i.imgur.com/TL3Pzyq.png" width = "375"></center>
<br>

><strong>Fig. 4.</strong> <strong>The relative composition of interaction outcomes differs with phylogenetic scale of the comparison.</strong> **a**. Interaction types broken down into within- and between-clade comparisons (RI = reciprocal invasibility; RNI = reciprocal non-invasibility; Asymm. = asymmetric dominance). Total strain pairings = 772 between 40 strains, with the 20 self-interactions and 8 no-data interactions removed. **b**. The estimated probability of a given interaction systematically varies with phylogenetic distance ($D_{p}$) between strain pairs (see Table 2 for GLM results). Graphs depict GLM results where self-interactions were not included.
> []I did not understand what you meant by "sub-clades" above, so either remove or define and then re-use here.

In binomial GLMs, the overall probability of an asymmetric outcome increased with genetic distance (Fig. S6b; Table S1). This is driven predominantly by the competitive superiority of *P. fluorescens* > *P. syringae*. When trait distances were included with genetic distance in interaction outcome GLMs, the effect of DG generally dropped out (Table S2). Within *P. fluorescens*, pairings between strains with larger differences in lag time more often resulted in asymmetric outcomes, while similarity in lag time predicted reciprocal invasibility (Table S2). *P. syringae* exhibited the same pattern except for rm instead of *l*, although residual variation was still explained by $D_{G}$, perhaps correlated with un-measured traits influencing competitiveness (Table S2).

Strains isolated from the same leaves were no more or less likely to display a particular competitive outcome than pairs of strains isolated from different leaves (Fisher’s exact tests, p > 0.3, data not shown), nor were winners more likely to have been at a higher abundance in leaves when they were sampled (Humphrey et al. 2014) (binomial sign test, $p = 0.45$, data not shown).

#### Competitive interaction network and intransitivity

Toxin-mediated facilitation. Our analysis of the patterns of pairwise interaction outcomes combined with the traits that predict them was expanded to include sets of three strains in order to evaluate the extent of intransitivity (non-hierarchical competitive asymmetries) within the bacterial interaction network. 8,203 trios were evaluated for facilitation based on the 641 pairs of strains that met the competitive asymmetry criteria (see Methods). Of these trios, 632 (7.7%) met the criteria whereby the inferior competitor was facilitated by the killing of the superior competitor by a third party to which the facilitated strain was resistant. Of all 40 strains, 39 were implicated at least once as a participant in a facilitative trio as one or more of the following roles: the facilitator, the knocked-out competitor, or the facilitated strain (‘A’, ‘B’, and ‘C’, respectively; Fig. 3a). Overall, 26 strains were facilitated, and 21 of these also served as the knocked-out competitor in a subset of the trios (Fig 4b, inset). Twelve of the 13 inhibitor-producing strains (all *P. fluorescens*) were implicated as facilitators (A strains) (Fig 4b, inset). In all but 6 of the 632 facilitative trios, the A strain was also exploitatively dominant to the C strain, even though such strains were resistant to their inhibitors (Fig 4b, arrows).

The number of trios in which a given strain played each role varied substantially among strains, with one strain in particular (43A) playing a disproportionate role as facilitator compared to other inhibitor-producing *P. fluorescens* (Fig. 3b). Three inhibitor-producing strains also played the role of the $B$ strain in trios including an $A$ strain to whose toxin it was susceptible (36A, 46A, 06B), and one *P. fluorescens* strain played all three roles (06B). For strains that exclusively played the $B$ and $C$ roles, the probability of playing a B role was weakly negatively correlated with the probability of playing a $C$ role (Pearson’s $r  = –0.30$ $[-0.56–0.12$ 95\% CI], $p=0.059$).


<!-- ******** -->
<!-- FIGURE 5 -->
<!-- ******** -->

<center>
<img src="https://i.imgur.com/amx5jXn.png" width = "600"></center>
<br>

><strong>Fig. 5. Prevalence of intransitive interactions in a *Pseudomonas* spp. interaction network.</strong> **a**. Types of interaction trios resulting in facilitation (left) or rock-paper-scissors (R–P–S) competitive asymmetries. $N$ = number of trios meeting the given criteria out of the total trios analyzed (see Methods). **b**. Frequency distributions of how often each strain played the facilitator ($A$), the knocked-out intermediate ($B$), or the facilitated ($C$). Several strains played multiple roles; strains in facilitative trios with as well as without toxic intermediates are indicated with black triangles to the left of the strain IDs. Panel (**b**) inset displays the distribution of the number of unique strains that played each combination of roles. 06B played all three. The probability of playing $A$, $B$, or $C$ roles in facilitative trios varied with (**c**) overall competitiveness, $C_{w}$, as well as (**d**) resistance, $C_{r}$. See Table 5 for GLM results.

The propensity towards $B$ vs. $C$ roles was explained by underlying differences in competitive fitness: the most facilitated strains (high C fraction) were among the least competitive (low $C_{w}$) in the populations, indicated by a negative correlation ($r  = –0.76$ [$|0.86–0.58|$ 95% CI], $p<10^{–5}$; Fig. 3c, Table 3). $B$ strains were intermediate relative to the entire range of $C_{w}$ values. Facilitator $A$ strains had consistently higher $C_{w}$, owing to the generally higher competitiveness of *P. fluorescens* strains (Fig. 1a–b). Resistance ($C_{r}$) was strongly positively correlated with the probability of being facilitated (Pearson’s $r = 0.57$ [$0.32–0.75$ 95% CI], $p<10^{–4}$; Fig. 1d, Table 3), and $C_{w}$ and $C{r}$ jointly considered in binomial GLMs independently ($C_{w}$ and $C_{r}$ are uncorrelated for *P. syringae*; Fig. 2a) contributed to variation in $B$ and $C$ roles (Fig. 3c–d, Table 3).

We also uncovered five trios whose mutual invasion asymmetries met the criteria for a rock–paper–scissor game out of the 9,604 possible trios of interactions evaluated (Fig. 3a). Nine unique strains were implicated in these trios. Each trio was comprised of distantly related *P. syringae* strains (mean $D_{G}$ between strains in R–P–S trios $= 0.118$ [$0.115–0.122$ 95% CI]; Fig S6b). Strains 21A and 24C were implicated in three R–P–S trios that included several unique third-party strains (Fig S6b).


### Discussion

Our analyses revealed major differences in both exploitative and interference competitiveness between the two major clades comprising the *Pseudomonas* spp. phyllosphere community of bittercress, as well as distinctions in how such competitiveness relates to underlying life history traits. Instead of trade-offs between life history traits, we uncovered distinct correlates of competitiveness that involved shorter lag phase duration for *P. fluorescens*, and higher maximum growth rates for *P. syringae*. Canonical trade-offs between rate and yield were not observed across either bacterial clade. Instead, we found a novel trade-off between lag phase and growth rate present only within *P. fluorescens*. Inhibition ability (i.e. interference competitive ability) did not trade-off with exploitative ability but was positively related to it. Such patterns suggest that the evolution of competitiveness may involve different components of life history for these bacterial lineages. 

Competitive dominance in within-clade competitions was largely explained by how dissimilar the life history traits were between the interacting pairs, and genetically more similar strains tended to display more similar life histories. 

Examining the community context of these interactions revealed that a modest but potentially important fraction of all three-way interactions result in the reversal of competitive outcomes that would otherwise lead to competitive exclusion of an inferior competitor. A small set of R–P–S interactions was discovered, which may lead to cyclical invasion dynamics. The intransitivity in our interaction network generally took the form of facilitation, whereby a toxic strain displaced a superior competitor thereby facilitating a resistant but weaker recipient. The community context of interference competition is important for predicting the exploitative capacity of resistant strains by buffering them from competitive exclusion, which alters the premium on exploitative capacity if a strain is also resistant. Such a dataset allows dissection of several dimensions of in vitro fitness exhibited by a natural community of phyllosphere *Pseudomonas* spp., and provides a platform for testing hypotheses about the mechanistic bases of competitive traits (e.g. toxin production and resistance). Together, this work helps build an understanding of how competitive traits might evolve in tandem with other life history traits in representatives from real communities that interact in nature.

#### Correlations between growth traits and competitiveness

Selection for increased exploitative competitive ability is expected to increase maximum growth rate, perhaps at the expense of growth efficiency, which can result in a tragedy of the commons whereby rapid but wasteful use of resources yields higher competitive ability (Pfeiffer et al. 2001; MacLean 2007). Consistent with this, overall exploitative competitive fitness (Cw) for *P. syringae* was positively correlated with maximum growth rate (rm) (Table 1). Notably, however, neither *P. syringae* nor *P. fluorescens* strains exhibited a canonical r–K trade-off: growth rate was positively correlated with yield in both minimal and rich media for *P. syringae*, and was uncorrelated with yield for *P. fluorescens* (Fig. 2, S5). In stark contrast to *P. syringae*, exploitative competitiveness for *P. fluorescens* was most correlated with having a shorter lag phase duration (1/L). Interestingly, shorter lag phase was negatively correlated with growth rate for *P. fluorescens*, but not for *P. syringae*. These findings indicate that exploitative ability for *P. syringae* and *P. fluorescens* likely are constrained by separate underlying life history characters. The unique correlational structure among these life history characters may constrain the evolution of exploitative ability for each clade in different ways.

We were surprised to uncover that maximum growth rate was correlated with a longer lag phase in *P. fluorescens*, as this pattern contradicts the traditional dichotomy between generally “fast” vs. “slow” life histories. *Escherichia coli* lines adapting to a glucose-limited environment exhibited coordinated increases in growth rate and shorter lag time after 10,000 generations (Vasi et al. 1994; Lenski et al. 1998). E. coli selected to persist in lag phase during periods of antibiotic stress incurred no pleiotropic cost of reduced maximum growth rate despite up to a 10-fold increase in lag time (Fridman et al. 2015). One explanation for our lag–growth correlation is that competitive fitness for *P. fluorescens* in the spatial microcosms we used may have more to do with space than resource use. Strains that preempt as much space as possible early on may reap the rewards of their territorial monopoly even at the expense of a decreased maximal growth rate. One potential mechanism is the production of exudates that prevent physical expansion of competitor cells. This explanation rests on an intuitive physiological trade-off between exudate production and cell replication, but explains both the premium on short lag as well as its later costs. Thus, a straightforward hypothesis is that lag phase is causally affected by the amount of exudate production and exudate production trades off with maximum growth rate. Lag phase has received renewed attention as a distinct component of microbial life cycles (Rolfe et al. 2012), and characterizing the physiology of cells during this phase may reveal the nature of its linkage with maximum growth rate.

*P. syringae* strains are less able to exert priority effects in the spatial context of our assays. However, if both strains were to compete in an unstructured environment where preemption of space was irrelevant, *P. syringae* strains with high growth rates might be expected to outcompete a variety of *P. fluorescens* strains with relatively lower growth rates (Fig 1). Thus, the negative correlation between lag phase and growth rate in *P. fluorescens* resembles a colonization–competition trade-off. Spatial priority effects arising from territoriality can theoretically provide a mechanism for maintenance of colonization–competition trade-offs that would otherwise lead to competitive exclusion (Edwards and Schreiber 2010). A colonization–competition trade-off underlies territoriality in *Vibrio* spp. based on the differential ability of clones to contest territory vs. disperse to new ephemeral habitats (Yawata et al. 2014). Either one of these mechanisms may contribute to the maintenance of diverse exploitation strategies in *P. fluorescens* across patchy and ephemeral leaf environments.

In light of these findings, we hypothesize that *P. fluorescens* may be a territorial species whose potential effect (among others) in the phyllosphere may be to exclude colonization by P. syringae strains. This prediction is consistent with the identity of *P. fluorescens* as a plant mutualist, although the evidence of this comes exclusively, to our knowledge, from studies of its indirect effects via plant defensive signaling or direct toxicity to pathogenic fungi following its colonizing of plant roots (Mendes et al. 2011; Hol et al. 2013). In addition to such indirect effects, the superior competitiveness of *P. fluorescens* vs. *P. syringae* noted in this study suggests that direct interactions may affect phyllosphere bacterial community assembly as well as plant disease risk from phytopathogenic isolates of *P. syringae*.

Growth of the laboratory strain Psm4326 in MM appeared attenuated compared to its four closest relatives in this strain collection (Fig. S1). This strain was also among the weakest competitors, both offensively and defensively. It is possible that this sensitivity arose as a consequence of domestication to the laboratory environment. 

#### Lack of trade-offs between exploitative and interference competition

Correlations between exploitative and interference competition may strongly depend on the underlying mechanisms of interference competition. Here, such an interference mechanism could range from direct injection of bacterial effectors via Type VI Secretion Systems (Decoin et al. 2014), the production of subversive growth-regulating diffusible N-acylhomoserine lactones (AHLs) or enzymes that quench these signals typically involved in quorum sensing (QS) (Dulla and Lindow 2009), or the production of diffusible toxins (e.g. bacteriocins or phage-derived proteins). Trade-offs between toxin production and toxin resistance with intrinsic growth rate are often presumed in models of intransitive competitive loops (e.g. Neumann and Jetschke 2010) and are necessary to permit co-existence of types.

Interestingly, inhibition ability ($C_t$) or resistance ($C_r$) did not trade-off with any of the life history traits measured in this study (Fig. 2a), which is consistent with recent findings that toxin induction did not affect in vitro life history measures in *P. fluorescens* (Garbeva et al. 2011). Instead, we detected a positive correlation between inhibitory ability ($C_t$) and overall exploitative competitiveness for *P. fluorescens*. Although unexpected, such a positive correlation is nevertheless intuitive: megacolonies invading a resident strain presumably must reach a critical size in order for any toxicity to be detectable if induction is either density dependent or if the toxic effects are concentration dependent. Cells may only reach such a critical density if their relative exploitative competitiveness enables them to do so, without which interference competitive ability is irrelevant.


#### Interactions between exploitative and interference competition

Irrespective of the underlying mechanisms of toxicity and resistance, the frequency of these traits in a community may have large indirect effects that generate intransitive asymmetries among diverse genotypes. Several theoretical studies on intransitivity call for increased empirical research in order to test the predictions that non-hierarchical competitiveness stabilizes diversity (Laird and Schamp 2006; Laird and Schamp 2008). Our results address this call by exploring some of the intransitive properties of an empirical interaction network measuring the joint outcomes of exploitative and interference competition. Instead of combining assortments of laboratory strains to explore community properties (e.g. Eisenhauer et al. 2012a; Eisenhauer et al. 2012b; Eisenhauer et al. 2013), we used a collection of natural isolates. Intransitivity in bacterial communities has been principally considered empirically with respect to bacteriocin production and resistance (Czaran et al. 2002; Kerr et al. 2002; Majeed et al. 2011).

In our interaction network, interference competition may equalize fitness differences between competitors that otherwise have asymmetric exploitative abilities (Fig. 3b). Many strains that were facilitated by an inhibitor producer to which they were resistant were also the strain whose inhibition facilitated another (Fig. 3c–d). Intransitive facilitation of the sort explored here is only possible with an intermediate frequency of toxin resistance expressed by *P. syringae* (Fig. 3d). The fact that resistance is not more common among *P. syringae* suggests a cost of resistance that did not manifest itself in the assays conducted in our study.

We show that the gains from facilitation predominantly go towards weaker resource competitors (Fig. 3c). Only in a small subset of the facilitation trios could the facilitated strain invade the producer. When the facilitated strain does not pose a competitive threat to the facilitator—as is the case most of the time here—the gains from facilitation may be short-lived. However, the overall effect of this degree of intransitive facilitation may be to prolong periods between exclusion/extinction events, elevating the diversity that is observable at any given point within the system (Laird and Schamp 2006; Laird and Schamp 2008; Laird 2014). The additional form of intransitivity detected in our study is a pair of extended trios that have R–P–S invasion asymmetries, which are predicted to lead to frequency-dependent or cyclical invasion dynamics (Laird 2014). This prediction is awaiting an empirical test, and this system presents an excellent opportunity for doing so.

These conceptual implications of intransitivity speak to the large degree of diversity (both genetic and phenotypic) apparently maintained within this *Pseudomonas* spp. community, despite pairwise competitive asymmetries between many strains. Due to the seasonal nature of the phyllosphere environment in bittercriss, extensive variation along an absolute fitness gradient might be expected, as spatial structure protects low fitness genotypes from global exclusion (Amarasekare 2003; Laird and Schamp 2008; Kryazhimskiy et al. 2012). Still, multiple distinct genotypes occurring within the same leaf is common (Fig. 1a), and the potential for local competitive interactions within or on leaf surfaces is large (Lindow and Brandl 2003).

The factors promoting co-existence may include competitive intransitivity mediated by exploitative and interference competition. An evolutionary consequence is that larger fitness differences may be required between genotypes or species for competitive exclusion to take place at the landscape scale (Cvijovic et al. 2018). This system is ripe for the modeling of how particular combinations of (1) life history traits and (2) inhibitor production and resistance traits can stabilize the prevalence of poor competitors embedded in non-hierarchical interaction networks. The joint implications of the maintenance of otherwise poor competitors for ecosystem-level traits such as productivity or trophic flow through food webs and the rate of adaptive evolution within species, remains a compelling topic for further study.

#### Conclusions

We found that competitive abilities of strains within a natural community of phyllosphere microbiome of *Pseudomonas* spp. varied between the two major clades present, *P. fluorescens* and *P. syringae*. Variation in competitiveness was best explained by distinct life history traits in each clade: shorter lag time in *P. fluorescens*, and increased maximum growth rate in *P. syringae*. The lack of expected life history trade-offs between growth rate and yield. The presence of different trait correlations that were distinct between clades illustrates the evolutionary lability of the relationships among these fundamental dimensions of competitive fitness. Conserved trait correlations within clades suggest that different life history strategies allows for clade persistence. Although speculative, the *P. fluorescens* clade may contain early colonizing strains that contest territory to a greater extent, which may serve to directly buffer against leaf colonization from potentially phytopathogenic *P. syringae*. In contrast, a high degree of inhibitor resistance among *P. syringae* may prevent local exclusion when spatial structure releases them from direct exploitative competition with *P. fluorescens*. Finally, the combination of exploitative and interference competition in this system potentially stabilizes co-existence of strains that might otherwise competitively exclude one another due to the fitness-equalizing force of inhibitor-mediated facilitation.


### References

Amarasekare, P. 2003. Competitive coexistence in spatially structured environments: a synthesis. Ecology Letters 6:1109–1122.

Baltrus, D. A., M. T. Nishimura, A. Romanchuk, J. H. Chang, M. S. Mukhtar, K. Cherkis, J. Roach, et al. 2011. Dynamic Evolution of Pathogenicity Revealed by Sequencing and Comparative Genomics of 19 Pseudomonas syringae Isolates. (D. S. Guttman, ed.)PLoS pathogens 7:e1002132.

Barrett, L. G., T. Bell, G. Dwyer, and J. Bergelson. 2011. Cheating, trade-offs and the evolution of aggressiveness in a natural pathogen population. Ecology Letters 14:1149–1157.

Bergstrom, A., J. T. Simpson, F. Salinas, B. Barre, L. Parts, A. Zia, A. N. Nguyen Ba, et al. 2014. A high-definition view of functional genetic variation from natural yeast genomes. Molecular Biology and Evolution 31:872–888.

Bronstein, J. L. 2009. The evolution of facilitation and mutualism. Journal of Ecology 97:1160–1170.

Brooker, R. W., F. T. Maestre, R. M. Callaway, C. L. Lortie, L. A. Cavieres, G. Kunstler, P. Liancourt, et al. 2008. Facilitation in plant communities: the past, the present, and the future. Journal of Ecology 96:18–34.

Cornforth, D. M., and K. R. Foster. 2013. Competition sensing: the social side of bacterial stress responses. Nature reviews Microbiology 11:285–293.

Cui, J., A. K. Bahrami, E. G. Pringle, G. Hernandez-Guzman, C. L. Bender, N. E. Pierce, and F. M. Ausubel. 2005. Pseudomonas syringae manipulates systemic plant defenses against pathogens and herbivores. Proceedings of the National Academy of Sciences of the United States of America 102:1791–1796.

Cui, J., G. Jander, L. R. Racki, P. D. Kim, N. E. Pierce, and F. M. Ausubel. 2002. Signals involved in Arabidopsis resistance to Trichoplusia ni caterpillars induced by virulent and avirulent strains of the phytopathogen Pseudomonas syringae. Plant physiology 129:551–564.

Czaran, T. L., R. F. Hoekstra, and L. Pagie. 2002. Chemical warfare between microbes promotes biodiversity. Proceedings of the National Academy of Sciences of the United States of America 99:786–790.

Decoin, V., C. Barbey, D. Bergeau, X. Latour, M. G. J. Feuilloley, N. Orange, and A. Merieau. 2014. A Type VI Secretion System Is Involved in Pseudomonas fluorescens Bacterial Competition. (J. Murillo, ed.)PloS one 9:e89411.

DeLong, J. P., and D. A. Vasseur. 2013. Linked exploitation and interference competition drives the variable behavior of a classic predator–prey system. Oikos 122:1393–1400.

Dulla, G. F. J., and S. E. Lindow. 2009. Acyl-homoserine lactone-mediated cross talk among epiphytic bacteria modulates behavior of Pseudomonas syringae on leaves. The ISME journal 3:825–834.

Dulla, G. F. J., K. V. Krasileva, and S. E. Lindow. 2010. Interference of quorum sensing in Pseudomonas syringae by bacterial epiphytes that limit iron availability. Environmental Microbiology 12:1762–1774.

Edwards, K. F., and S. J. Schreiber. 2010. Preemption of space can lead to intransitive coexistence of competitors. Oikos 119:1201–1209.

Eisenhauer, N., E. Materne, S. Scheu, and A. Jousset. 2013. Evolutionary history predicts the stability of cooperation in microbial communities. Nature communications 4:1–7.

Eisenhauer, N., S. Scheu, and A. Jousset. 2012a. Bacterial Diversity Stabilizes Community Productivity. (S. Bertilsson, ed.)PloS one 7:e34517.

Eisenhauer, N., W. Schulz, S. Scheu, and A. Jousset. 2012b. Niche dimensionality links biodiversity and invasibility of microbial communities. (M. Pfrender, ed.)Functional Ecology 27:282–288.

Foster, K. R., and T. Bell. 2012. Competition, not cooperation, dominates interactions among culturable microbial species. Current biology : CB 22:1845–1850.

Frank, S. A. 2010. The trade-off between rate and yield in the design of microbial metabolism. Journal of evolutionary biology 23:609–613.

Frean, M., and E. R. Abraham. 2001. Rock-scissors-paper and the survival of the weakest. Proceedings of the Royal Society B: Biological Sciences 268:1323–1327.

Fridman, O., A. Goldberg, I. Ronin, N. Shoresh, and N. Q. Balaban. 2015. Optimization of lag time underlies antibiotic tolerance in evolved bacterial populations. Nature 513:418–421.

Garbeva, P., O. Tyc, M. N. P. Remus-Emsermann, A. van der Wal, M. Vos, M. Silby, and W. de Boer. 2011. No apparent costs for facultative antibiotic production by the soil bacterium Pseudomonas fluorescens Pf0-1. PloS one 6:e27266.

Groen, S. C., N. K. Whiteman, A. K. Bahrami, A. M. Wilczek, J. Cui, J. A. RUSSELL, A. Cibrian-Jaramillo, et al. 2013. Pathogen-triggered ethylene signaling mediates systemic-induced susceptibility to herbivory in Arabidopsis. The Plant cell 25:4755–4766.

Hibbing, M. E., C. Fuqua, M. R. Parsek, and S. B. Peterson. 2009. Bacterial competition: surviving and thriving in the microbial jungle. Nature reviews Microbiology 8:15–25.

Hol, W. H. G., T. M. Bezemer, and A. Biere. 2013. Getting the ecology into interactions between plants and the plant growth-promoting bacterium Pseudomonas fluorescens. Frontiers in plant science 4:81.

Humphrey, P. T., T. T. Nguyen, M. M. Villalobos, and N. K. Whiteman. 2014. Diversity and abundance of phyllosphere bacteria are linked to insect herbivory. Molecular Ecology 23:1497–1515.

Huynh, T. V., D. Dahlbeck, and B. J. Staskawicz. 1989. Bacterial blight of soybean: regulation of a pathogen gene determining host cultivar specificity. Science (New York, NY) 245:1374–1377.

Jasmin, J.-N., and C. Zeyl. 2012. Life-history evolution and density-dependent growth in experimental populations of yeast. Evolution 66:3789–3802.

Kerr, B., M. A. Riley, M. W. Feldman, and B. J. M. Bohannan. 2002. Local dispersal promotes biodiversity in a real-life game of rock-paper-scissors. Nature 418:171–174.

Kinkel, L. L., and S. E. Lindow. 1993. Invasion and Exclusion among Coexisting Pseudomonas syringae Strains on Leaves. Applied and environmental microbiology 59:3447–3454.

Kirkup, B. C., and M. A. Riley. 2004. Antibiotic-mediated antagonism leads to a bacterial game of rock-paper-scissors in vivo. Nature 428:412–414.

Kryazhimskiy, S., D. P. Rice, and M. M. Desai. 2012. Population subdivision and adaptation in asexual populations of Saccharomyces cerevisiae. Evolution 66:1931–1941.

Laird, R. A. 2014. Population interaction structure and the coexistence of bacterial strains playing “rock-paper-scissors.” Oikos 123:472–480.

Laird, R. A., and B. S. Schamp. 2006. Competitive intransitivity promotes species coexistence. American Naturalist 168:182–193.

Laird, R. A., and B. S. Schamp. 2008. Does local competition increase the coexistence of species in intransitive networks? Ecology 
89:237–247.

Lankau, R. A., and S. Y. Strauss. 2007. Mutual feedbacks maintain both genetic and species diversity in a plant community. Science (New York, NY) 317:1561–1563.

Lenski, R. E., and M. A. Riley. 2002. Chemical warfare from an ecological perspective. Proceedings of the National Academy of Sciences of the United States of America 99:556–558.

Lenski, R. E., J. A. Mongold, P. D. Sniegowski, M. Travisano, F. Vasi, P. J. Gerrish, and T. M. Schmidt. 1998. Evolution of competitive fitness in experimental populations of E. coli: what makes one genotype a better competitor than another? Antonie van 
Leeuwenhoek 73:35–47.

Lindow, S. E., and M. T. Brandl. 2003. Microbiology of the phyllosphere. Applied and environmental microbiology 69:1875–1883.

Luckinbill, L. S. 1978. r and K selection in experimental populations of Escherichia coli. Science (New York, NY) 202:1201–1203.

Luckinbill, L. S. 1979. Selection and the r/K continuum in experimental populations of protozoa. American Naturalist 427–437.

Ma, A., D. Lv, X. Zhuang, and G. Zhuang. 2013. Quorum quenching in culturable phyllosphere bacteria from tobacco. International journal of molecular sciences 14:14607–14619.

MacLean, R. C. 2007. The tragedy of the commons in microbial populations: insights from theoretical, comparative and experimental studies. Heredity 100:471–477.

Majeed, H., O. Gillor, B. Kerr, and M. A. Riley. 2011. Competitive interactions in Escherichia coli populations: the role of bacteriocins. The ISME journal 5:71–81.

McIntire, E. J. B., and A. Fajardo. 2013. Facilitation as a ubiquitous driver of biodiversity. The New phytologist 201:403–416.

Mendes, R., M. Kruijt, I. de Bruijn, E. Dekkers, M. van der Voort, J. H. M. Schneider, Y. M. Piceno, et al. 2011. Deciphering the Rhizosphere Microbiome for Disease-Suppressive Bacteria. Science (New York, NY) 332:1097–1100.

Mitri, S., and K. Richard Foster. 2013. The Genotypic View of Social Interactions in Microbial Communities. Annual review of genetics 47:247–273.

Monier, J.-M., and S. E. Lindow. 2005. Spatial organization of dual-species bacterial aggregates on leaf surfaces. Applied and environmental microbiology 71:5484–5493.

Morris, C. E., D. C. Sands, B. A. Vinatzer, C. Glaux, C. Guilbaud, A. Buffière, S. Yan, et al. 2008. The life history of the plant pathogen Pseudomonas syringae is linked to the water cycle. The ISME journal 2:321–334.

Mudgett, M. B., and B. J. Staskawicz. 1999. Characterization of the Pseudomonas syringae pv. tomato AvrRpt2 protein: demonstration of secretion and processing during bacterial pathogenesis. Molecular microbiology 32:927–941.

Neumann, G. F., and G. Jetschke. 2010. Evolutionary classification of toxin mediated interactions in microorganisms. BioSystems 99:155–166.

Novak, M., T. Pfeiffer, R. E. Lenski, U. Sauer, and S. Bonhoeffer. 2006. Experimental tests for an evolutionary trade-off between growth rate and yield in E. coli. The American naturalist 168:242–251.

Ohgushi, T., O. Schmitz, and R. D. Holt. 2012. Trait-Mediated Indirect Interactions. Cambridge University Press.

Oksanen, J., F. G. Blanchet, R. Kindt, P. Legendre, P. R. Minchin, R. B. OHara, G. L. Simpson, et al. 2012. vegan: Community Ecology Package.

Ostrowski, E. A., D. E. Rozen, and R. E. Lenski. 2005. Pleiotropic effects of beneficial mutations in Escherichia coli. Evolution 59:2343–2352.

Pfeiffer, T., S. Schuster, and S. Bonhoeffer. 2001. Cooperation and competition in the evolution of ATP-producing pathways. Science (New York, NY) 292:504–507.

Quiñones, B., G. Dulla, and S. E. Lindow. 2005. Quorum sensing regulates exopolysaccharide production, motility, and virulence in Pseudomonas syringae. Molecular plant-microbe interactions : MPMI 18:682–693.

Revelle, W. 2012. psych: Procedures for Personality and Psychological Research. Northwestern University, Evanston, Illinois, USA.

Rojas-Echenique, J., and S. Allesina. 2011. Interaction rules affect species coexistence in intransitive networks. Ecology 92:1174–1180.

Rolfe, M. D., C. J. Rice, S. Lucchini, C. Pin, A. Thompson, A. D. S. Cameron, M. Alston, et al. 2012. Lag Phase Is a Distinct Growth Phase That Prepares Bacteria for Exponential Growth and Involves Transient Metal Accumulation. Journal of bacteriology 194:686–701.

Sinervo, B., and C. M. Lively. 1996. The rock-paper-scissors game and the evolution of alternative male strategies. Nature 380:240–243.

Vasi, F., M. Travisano, and R. E. Lenski. 1994. Long-term experimental evolution in Escherichia coli. II. Changes in life-history traits during adaptation to a seasonal environment. American Naturalist 432–456.

Warringer, J., E. Zörgö, F. A. Cubillos, A. Zia, A. Gjuvsland, J. T. Simpson, A. Forsmark, et al. 2011. Trait Variation in Yeast Is Defined by Population History. (L. Kruglyak, ed.)PLoS genetics 7:e1002111.

Wilson, M., and S. E. Lindow. 1994a. Coexistence among Epiphytic Bacterial Populations Mediated through Nutritional Resource Partitioning. Applied and environmental microbiology 60:4468–4477.

Wilson, M., and S. E. Lindow. 1994b. Ecological Similarity and Coexistence of Epiphytic Ice-Nucleating (Ice) Pseudomonas syringae Strains and a Non-Ice-Nucleating (Ice) Biological Control Agent. Applied and environmental microbiology 60:3128–3137.

Yawata, Y., O. X. Cordero, F. Menolascina, J. H. Hehemann, M. F. Polz, and R. Stocker. 2014. Competition-dispersal tradeoff ecologically differentiates recently speciated marine bacterioplankton populations. Proceedings of the National Academy of Sciences 111:5622–5627.


### Acknowledgements

<hr>

### Supplemental Information

#### Supplemental Figures


<center>
<img src="https://i.imgur.com/nx1AZMl.png" width = "600"></center>
<br>

><strong>Fig. S1. Growth cycle profiles of 40 *Pseudomonas* spp. strains measured individually in minimal media <em>in vitro</em></strong>. Overlain individual growth curves are shown grouped by nearest phylogenetic neighboring strains (see Fig. 1, main text). Strain 08B is sister to *P. syringae* clade. 4326 = laboratory strain *P. syringae* pv. maculicola str. ES4326.

<br>


<center>
<img src="https://i.imgur.com/Pba01IX.png" width = "500"></center>
<br>

><strong>Fig. S2.</strong> <strong>Life history trait correlations exhibited by *P. syringae* and *P. fluorescens* strains.</strong> **a-b**. Correlations between (**a**) canonical growth-related traits and between growth-related traits and (**b**) competitive metrics. **c**. Correlations between competitive metrics. Linear fits only displayed for lines with Pearson’s $r$ with FDR-corrected $p < 0.05$ (see Results, main text).


<br>

<center>
<img src="https://i.imgur.com/u3ywT7h.png" width = "400"></center>
<br>

><strong>Fig. S3.</strong> <strong>Morphological variation observed during pairwise inhibition assays reflects strain-specific induction of motility and inhibition phenotypes in *P. fluorescens*.</strong> **a-c**. Photos of interaction plates against *P. syringae* resident strains against which *P. fluorescens* strain 43A adopted distinct megacolony morphologies (white arrows). Black arrows denote invading strains producing tox+ halos against the resident strains. **d-f**. close-ups of smooth morph (SM), wrinkly spreader-like (WS-like), and the smooth spreader (SS) morphs from above. **g**. Close-up of putative growth facilitation of non-focal strain via secretion of effectors from 43A, arising from either cell cycle modification by secreted regulators, or competitive release via killing of resident strain. H. Phylogenetic specificity of morphological induction and its relationship with toxicity for strain 43A. Phylogenetic tree from Fig. 1 (main text). Scale bars in **a–c** = 1 cm; **d–f** = 0.50 cm; **g** = 0.25 cm.

