---
title: "Competition manuscript"
author: "Parris T Humphrey"
bibliography: competition.bib
csl: evolution.csl
---

## Competitive hierarchies, antibiosis, and the distribution of bacterial life history traits in a microbiome

#### Parris T. Humphrey<sup>1,2,3*</sup>, Trang N. Nguyen<sup>4</sup>, Noah K. Whiteman<sup>2</sup>

<sup>1</sup>Department of Organismic & Evolutionary Biology, Harvard University, Cambridge, MA, USA<br>
<sup>2</sup>Department of Integrative Biology, University of California, Berkeley, CA, USA<br>
<sup>3</sup>Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, USA<br>
<sup>4</sup>Department of Plant Sciences, University of Arizona, Tucson, AZ, USA

**\*Correspondence and requests for materials** should be addressed to P.T.H. ([phumphrey@g.harvard.edu](phumphrey@g.harvard.edu))


<hr>

### Abstract

Microbiome manipulation depends on understanding how species interact ecologically within these communities. Can ecological outcomes be predicted from bacterial life history traits? We addressed this by studying interaction hierarchies in a community of 40 *Pseudomonas* spp. bacterial isolates from bittercress leaves (Brassicaceae: *Cardamine cordifolia* A. Gray). Pairwise competition experiments revealed competitive dominance of *P. fluorescens* over *P. syringae* that was predicted by phylogeny. *P. fluorescens* strains were superior competitors and more often produced antibiotics to which few *P. syringae* strains were resistant. *P. syringae* strains with higher growth rates won more contests, while *P. fluorescens* strains with shorter lag times and lower growth rates won more contests. Many competitive outcomes among *P. syringae* strains were reversed by *P. fluorescens* inhibitors because indirect benefits accrued to less competitive strains. *P. fluorescens* strains frequently changed competitive outcomes, suggesting a critical role of this species in structuring plant microbiome communities.

**Keywords:** interference competition; species interactions; *Pseudomonas*; co-existence; facilitation; microbiome

**Author contributions:** PTH and NKW designed the study; PTH and TNN collected the data; PTH and NKW wrote the manuscript.

<hr>

The ecological forces shaping bacterial microbiome community structure, particularly in mammals are difficult to characterize, given the diversity and relatively uncultivable nature of these taxa. Plants are exceptional in this regard because they possess a highly cultivable microbiome and therefore have potential to serve as models for understanding microbiome ecology and evolution more generally. Moreover, plant growth-promoting bacterial (PGPB) formulations are being deployed in agriculture, and understanding and even predicting ecological outcomes among common species in these artificial communities would be of practical value.

Competition may be the principle ecological force shaping microbial community structure [@Foster12a; @Coyte19a], yet distinct forms of competition can operate within communities: competition for shared resources and interference with another species’ ability to do so [@Case74a]. In addition to structuring microbiome communities, competition of both types is also a source of natural selection [@Hibbing10a; @Cornforth13a; @Mitri13a]. Teasing apart how exploitative and interference competition interact in a community context remains a challenge [@Amarasekare03a; @Coyte15a; @Delong13a].

As diversity increases, the number of possible indirect interactions in the community scales faster than the number of direct interactions. Accordingly, a species may benefit from additional competitors if the net indirect effects dampen direct competition faced by other species [@Levine76a; @Lawlor79a; @Stone91a; @Miller96a; @Wooton94a]. Such indirect facilitation [@Levine99a] has not been well explored in microbiomes.

Species-rich communities are more likely to harbor members with traits that have a large ecological impact [@Banerjee18a]. In microbial communities, strains that secrete diffusible antibiotics, resource substrates, or signaling molecules can alter the fitness of non-producers [@Lee10a; @Gutierrez19a]. By selecting for more specialized traits involved in resistance or metabolite uptake, these secretions can upend competitive hierarchies and trait distributions that would otherwise occur based on canonical competitive fitness traits. If microbial taxa with large indirect impacts are common or rare is unclear [@Banerjee18a]. Leaf-dwelling (phyllosphere) bacteria secrete compounds altering growth and survival of nearby bacteria [@Lindow03a; Quinones05a; Dulla09a; Dulla10a] and can co-localize on the leaf surface and interior [Monier05a]. Thus, there is potential for direct and indirect interactions between competing bacteria to affect community assembly and steady-state patterns of diversity in the phyllosphere.

Finally, competition need not be purely hierarchical: intransitive loops may arise in species-rich communities whereby numerical dominance cycles at local spatial scales, resulting in community stability [@Rojas-Echenique11a; @Kerr02a]. Even modest intransitivity can buffer against extinction [@Laird06a; @Laird08a; @Rojas-Echenique11a; @Laird14a] and the degree of intransitivity can shape species diversity [@Reichenbach07a]. Although intransitivity occurs in microbial systems in the laboratory [@Kerr02a; @Kelsic15a], its occurrence in native microbiome communities is not well characterized [@Lankau11a; @Godoy17a]. 

To address the various gaps highlighted above, here we (1) characterized life history trait variances and co-variances among isolates of a wild endophytic microbiome meta-community, (2) examined how this related to competitive interaction networks, and (3) measured how intransitive competitive asymmetries among strains promoted co-existence. We used a natural community of endophytic *Pseudomonas* spp. endophytic bacteria from a native plant (bittercress; Brassicaceae: *Cardamine cordifolia* A. Gray), including a diversity of strains of *P. syringae* and *P. fluorescens*. 

### Results

#### Competitive outcomes.
Pairwise soft-agar invasion assays revealed that the competitive abilities of *P. fluorescens* strains was superior to *P. syringae* strains across all measures of competitiveness (Fig. 1; Fig. 2a). Nearly 99% of pairings between clades resulted in asymmetric dominance of <i>P. fluorescens</i> over <i>P. syringae</i> (99% Asym.; Fig. S3; Tables S1, S2). Within <i>P. fluorescens</i>, the proportion of reciprocally non-invasible (RNI) pairings was significantly higher compared to within <i>P. syringae</i> pairings (Fig. S3; Tables S1, S2).


#### Interference competition.
Of the 40 strains assayed, 13 (all *P. fluorescens*) produced halos surrounding some subset of the resident strains they invaded (antibiosis), indicating the production of diffusible inhibitors (Fig. 1; Fig. S2). Putative resistance to these *P. fluorescens* secretions was common among strains from both clades (Fig. 2a-b).

Although we scored inhibition as a binary trait (see Methods), the strength and extent of inhibition varied among strain pairings, examples of which are shown in Fig. S4a–c (black arrows). Mean inhibition index ($C_{t}$) was 0.20, although two strains inhibited only one other, and *P. fluorescens* strain 03A failed to inhibit any strain (Fig. 1). Inhibition interactions were predominately restricted to *P. fluorescens* pairings with *P. syringae*, although four *P. fluorescens* strains were susceptible to inhibition by two of the toxic strains (43A, 34A; Fig. 1). Resistance to toxin producers in *P. syringae* was variable, although the mean value was high at 0.72 (Fig. 2b).

#### Life history correlates of competitiveness.
Offense ($C_{o}$) and defense ($C_{d}$) were strongly positively correlated overall with linear slopes near 1 (Fig. 2a; Fig. S4c). However, there is a dense and narrow band of strains with Co between 0.2 and 0.25 that showed a wide range of Cd values between 0.03 and 0.6 (Fig. S4c). Within this range, offense and defense appear uncorrelated, which may indicate that a saturating function fits the data better. However, a second-order polynomial regression did not improve fit (log likelihood ratio test; $\chi^{2} = 0.8$, $p > 0.2$, data not shown). Both offense and defense compared against the composite measure Cw, yielded similar near-1 slopes but with slightly offset elevations between the clades (Fig. S4c).

<!-- ******** -->
<!-- FIGURE 1 -->
<!-- ******** -->

<!--<center><img src="https://i.imgur.com/HtPIGmq.png" width = "600"></center><br>-->
<center><img src="https://i.imgur.com/aisfE3d.png" width = "800"></center><br>

><strong>Fig. 1. Phylogenetic distribution of life history trait variation within a *Pseudomonas* spp. community.</strong><strong>a</strong>. Life history components are maximum growth rate ($r_{m}$), lag phase ($l$), maximum yield ($K$), derived from individual microcosm growth experiments; and components of offensive ($C_{o}$), defensive ($C_{d}$), overall ($C_{w}$) competitiveness, resistance to toxicity ($C_{r}$), and toxicity ($C_{t}$), measured on derived from a pairwise competitive interaction network (see Methods). Column $z$-score of each trait value indicated by color. <strong>b</strong>. Smoothed frequency distributions of trait values for each measured trait by clade (*P. fluorescens* and *P. syringae*). Mean ($\mu$) estimates per clade with $\pm 2\times$ standard errors depicted to the right of the curves. Note the $x$-axis value scale modifiers to the right of the axis labels. (**c**) Pairwise correlations and principle component analysis (PCA) (**d**) of six traits reflect dissimilarities between clades, as well reveal the correlational structure among traits across *Pseudomonas* spp. Correlations with text values reflect magnitude of each Pearson’s $r$ where the FDR corrected $p < 0.05$; comparisons with FDR-corrected $p<0.10$ are italicized. **d**. PCA 95% envelopes per clade depicted as solid or dashed ellipses. Dots are labeled with strain IDs. Individual trait vector loadings are in blue for resource use traits and orange for interference traits.)

Competition and growth traits revealed opposing correlational patterns between the clades: $C_{w}$ was positively correlated with rm for *P. syringae* (model $F_{3,21} = 11.5$, adjusted $r^{2} = 0.57$; Table 1), but negatively with both $r$ and $L$ for *P. fluorescens*. When analyzed in a multiple regression, the correlation with $r$ disappeared, revealing a strong negative relationship with $L$ as the sole driver of variation in $C_{w}$ for *P. fluorescens* (model $F_{3,21} = 15.3$, adjusted $r^{2} = 0.77$; Table 1). $C_{w}$ in *P. syringae*, on the other hand, was solely explained by $r$; Table 1).

PCA of $C_{w}$, $C_{r}$, and $C_{t}$ with the three growth traits revealed largely non-overlapping 95% confidence ellipses for the two clades (Fig. 2b). The first two principal components together explained 72.5% of the variation in the data. The loading vectors of competitiveness and lag phase were in opposing directions, indicating a negative correlation, while those for competitiveness and inhibitory capacity are largely colinear, indicating a positive correlation (Fig. 2b). The loading for resistance, $C_{r}$, is nearly linear to lag phase duration, a relationship not apparent in the pair-wise correlation analysis in Fig. 2a. Strain 08B—tentatively categorized as *P. syringae* in this analysis but phylogenetically sister to that clade, falls beyond the 95% confidence ellipses for both named clades (Fig. 2b). 

#### Competitive interaction network and intransitivity.
Toxin-mediated facilitation. Our analysis of the patterns of pairwise interaction outcomes combined with the traits that predict them was expanded to include sets of three strains in order to evaluate the extent of intransitivity (non-hierarchical competitive asymmetries) within the bacterial interaction network. 8,203 trios were evaluated for facilitation based on the 641 pairs of strains that met the competitive asymmetry criteria (see Methods). Of these trios, 632 (7.7%) met the criteria whereby the inferior competitor was facilitated by the killing of the superior competitor by a third party to which the facilitated strain was resistant. Of all 40 strains, 39 were implicated at least once as a participant in a facilitative trio as one or more of the following roles: the facilitator, the knocked-out competitor, or the facilitated strain (‘A’, ‘B’, and ‘C’, respectively; Fig. 3a). Overall, 26 strains were facilitated, and 21 of these also served as the knocked-out competitor in a subset of the trios (Fig 4b, inset). Twelve of the 13 inhibitor-producing strains (all *P. fluorescens*) were implicated as facilitators (A strains) (Fig 4b, inset). In all but 6 of the 632 facilitative trios, the A strain was also exploitatively dominant to the C strain, even though such strains were resistant to their inhibitors (Fig 4b, arrows).

The number of trios in which a given strain played each role varied substantially among strains, with one strain (43A) playing a disproportionate role as facilitator compared to other inhibitor-producing *P. fluorescens* (Fig. 3b). Three inhibitor-producing strains also played the role of the $B$ strain in trios including an $A$ strain to whose toxin it was susceptible (36A, 46A, 06B), and one *P. fluorescens* strain played all three roles (06B). For strains that exclusively played the $B$ and $C$ roles, the probability of playing a B role was weakly negatively correlated with the probability of playing a $C$ role (Pearson’s $r  = –0.30$ $[-0.56–0.12$ 95\% CI], $p=0.059$).

<!-- ******** -->
<!-- FIGURE 2 -->
<!-- ******** -->

<center>
<img src="https://i.imgur.com/amx5jXn.png" width = "500"></center>
<br>

><strong>Fig. 2. Prevalence of intransitive interactions in a *Pseudomonas* spp. interaction network.</strong> **a**. Types of interaction trios resulting in facilitation (left) or rock-paper-scissors (R–P–S) competitive asymmetries. $N$ = number of trios meeting the given criteria out of the total trios analyzed (see Methods). **b**. Frequency distributions of how often each strain played the facilitator ($A$), the knocked-out intermediate ($B$), or the facilitated ($C$). Several strains played multiple roles; strains in facilitative trios with as well as without toxic intermediates are indicated with black triangles to the left of the strain IDs. Panel (**b**) inset displays the distribution of the number of unique strains that played each combination of roles. 06B played all three. The probability of playing $A$, $B$, or $C$ roles in facilitative trios varied with (**c**) overall competitiveness, $C_{w}$, as well as (**d**) resistance, $C_{r}$. See Table 5 for GLM results.

The propensity towards $B$ vs. $C$ roles was explained by underlying differences in competitive fitness: the most facilitated strains (high C fraction) were among the least competitive (low $C_{w}$) in the populations, indicated by a negative correlation ($r  = –0.76$ [$|0.86–0.58|$ 95% CI], $p<10^{–5}$; Fig. 3c, Table 3). $B$ strains were intermediate relative to the entire range of $C_{w}$ values. Facilitator $A$ strains had consistently higher $C_{w}$, owing to the generally higher competitiveness of *P. fluorescens* strains (Fig. 1a–b). Resistance ($C_{r}$) was strongly positively correlated with the probability of being facilitated (Pearson’s $r = 0.57$ [$0.32–0.75$ 95% CI], $p<10^{–4}$; Fig. 1d, Table 3), and $C_{w}$ and $C{r}$ jointly considered in binomial GLMs independently ($C_{w}$ and $C_{r}$ are uncorrelated for *P. syringae*; Fig. 2a) contributed to variation in $B$ and $C$ roles (Fig. 3c–d, Table 3).

We also uncovered five trios whose mutual invasion asymmetries met the criteria for a rock–paper–scissor game out of the 9,604 possible trios of interactions evaluated (Fig. 3a). Nine unique strains were implicated in these trios. Each trio was comprised of distantly related *P. syringae* strains (mean $D_{G}$ between strains in R–P–S trios $= 0.118$ [$0.115–0.122$ 95% CI]; Fig S6b). Strains 21A and 24C were implicated in three R–P–S trios that included several unique third-party strains (Fig S6b).

### Discussion

#### Overiew
We discovered major differences in both exploitative and interference competitiveness between the two major clades comprising the *Pseudomonas* spp. phyllosphere community of bittercress, as well as distinctions in how such competitiveness relates to underlying life history traits. Instead of trade-offs between life history traits, we uncovered distinct correlates of competitiveness that involved shorter lag phase duration for *P. fluorescens*, and higher maximum growth rates for *P. syringae*. Canonical trade-offs between rate and yield were not observed across either bacterial clade. Instead, we found a trade-off between lag phase and growth rate present only within *P. fluorescens*. Inhibition ability (interference competitive ability) did not trade-off with exploitative ability but was positively related to it. Such patterns suggest that the evolution of competitiveness may involve distinct components of life history in these bacterial lineages. 

Competitive dominance in within-clade competitions was largely explained by how dissimilar the life history traits were between the interacting pairs, and genetically more similar strains tended to display more similar life histories (Fig. 1). 

Examining the community context of these interactions revealed that a modest fraction of all three-way interactions resulted in the reversal of competitive outcomes that would otherwise lead to competitive exclusion of an inferior competitor. A small set of R–P–S interactions was discovered, which may lead to cyclical invasion dynamics. More interestingly, the predominant form of intransitivity in our interaction network took the form of facilitation, whereby a toxic strain displaces a superior competitor and thereby facilitates a resistant but weaker recipient. Thus, the community context of interference competition is important for predicting the outcome of competitive pairings whose outcome typically rests on exploitative capacity.

Such a dataset allows dissection of several dimensions of in vitro fitness exhibited by a natural community of phyllosphere *Pseudomonas* spp., and provides a platform for testing hypotheses about the mechanistic bases of competitive traits (e.g. toxin production and resistance). Together, this work helps build an understanding of how competitive traits might evolve in tandem with other life history traits in representatives from real communities that interact in nature.

#### Correlations between growth traits and competitiveness.
Selection for increased exploitative competitive ability is expected to increase maximum growth rate, perhaps at the expense of growth efficiency, which can result in a tragedy of the commons whereby rapid but wasteful use of resources yields higher competitive ability (Pfeiffer et al. 2001; MacLean 2007). Consistent with this, overall exploitative competitive fitness ($C_{w}$) for *P. syringae* was positively correlated with maximum growth rate, $r$ (Table 1). Notably, however, neither *P. syringae* nor *P. fluorescens* strains exhibited a trade-off with maximum yield, $K$: growth rate was positively correlated with yield for *P. syringae*, and uncorrelated for *P. fluorescens* (Fig. S2).

In contrast to *P. syringae*, exploitative competitiveness for *P. fluorescens* was most correlated with having a shorter lag phase duration (1/L). Shorter lag phase was negatively correlated with growth rate for *P. fluorescens*, but not for *P. syringae*. These findings indicate that exploitative ability for *P. syringae* and *P. fluorescens* are constrained by separate underlying life history characters, as further evidenced by the distinct clade-level clustering of strains in a multivariate analysis (Fig. 1d). Further, the degree of trait dispersion was greater in *P. syringae*, despite similar levels of average genetic divergence among strains. The unique levels of trait variation and correlational structure among traits may indicate that the genetic avenues by which competitive ability evolves in each clade are highly distinct.

We were surprised to uncover that maximum growth rate was correlated with a longer lag phase in *P. fluorescens*, as this pattern contradicts the traditional dichotomy between generally “fast” vs. “slow” life histories and contrasts with patterns observed in microbial evolution experiments [@Vasi94a]. 

*Escherichia coli* lines adapting to a glucose-limited environment exhibited coordinated increases in growth rate and shorter lag time after 10,000 generations (Vasi et al. 1994; Lenski et al. 1998). *E. coli* selected to persist in lag phase during periods of antibiotic stress incurred no pleiotropic cost of reduced maximum growth rate despite up to a 10-fold increase in lag time (Fridman et al. 2015).

One explanation for the positive $r-K$ correlation observed for *P. fluorescens* is that competitive fitness for these strains in the spatial microcosms may have more to do with space than resource use. Strains that preempt as much space as possible early on may reap the rewards of their territorial monopoly even at the expense of a decreased maximal growth rate. One potential mechanism is the production of exudates that prevent physical expansion of competitor cells. This explanation rests on an intuitive physiological trade-off between exudate production and cell replication, but explains both the premium on short lag as well as its later costs. Thus, a straightforward hypothesis is that lag phase is causally affected by the amount of exudate production and exudate production trades off with maximum growth rate. Lag phase has received renewed attention as a distinct component of microbial life cycles (Rolfe et al. 2012), and characterizing the physiology of cells during this phase may reveal the nature of its linkage with maximum growth rate.

#### Priority effects. 
*P. syringae* strains are less able to exert priority effects in the spatial context of our assays. However, if both strains were to compete in an unstructured environment where preemption of space was irrelevant, *P. syringae* strains with high growth rates might be expected to outcompete a variety of *P. fluorescens* strains with relatively lower growth rates (Fig 1). Thus, the negative correlation between lag phase and growth rate in *P. fluorescens* resembles a colonization–competition trade-off. Spatial priority effects arising from territoriality can provide a mechanism for maintenance of colonization–competition trade-offs that would otherwise lead to competitive exclusion (Edwards and Schreiber 2010). A colonization–competition trade-off underlies territoriality in *Vibrio* spp. based on the differential ability of clones to contest territory vs. disperse to new ephemeral habitats (Yawata et al. 2014). Either one of these mechanisms may contribute to the maintenance of diverse exploitation strategies in *P. fluorescens* across patchy and ephemeral leaf environments.

In light of these findings, we hypothesize that *P. fluorescens* may be a territorial species whose potential effect in the phyllosphere may be to exclude colonization by *P. syringae* strains. This is consistent with the identity of *P. fluorescens* as a plant mutualist, although the evidence of this comes exclusively, to our knowledge, from studies of its indirect effects via plant defensive signaling or direct toxicity to pathogenic fungi following its colonizing of plant roots (Mendes et al. 2011; Hol et al. 2013). In addition to such indirect effects, the superior competitiveness of *P. fluorescens* vs. *P. syringae* found in our study suggests that direct interactions may affect phyllosphere bacterial community assembly and plant disease risk from phytopathogenic isolates of *P. syringae*.

Growth of the laboratory strain Psm4326 in MM was attenuated compared to its four closest relatives in this strain collection (Fig. S1). Psm4326 was also among the weakest competitors, both offensively and defensively. It is possible that this sensitivity arose as a consequence of domestication to the laboratory environment. 

#### Lack of trade-offs between exploitative and interference competition.
Correlations between exploitative and interference competition may depend on the underlying mechanism of interference competition. Here, such an interference mechanism could range from direct injection of bacterial effectors via Type VI Secretion Systems (Decoin et al. 2014), the production of subversive growth-regulating diffusible N-acylhomoserine lactones (AHLs) or enzymes that quench these signals typically involved in quorum sensing (QS) (Dulla and Lindow 2009), or the production of diffusible toxins (e.g. bacteriocins or phage-derived proteins). Trade-offs between toxin production and toxin resistance with intrinsic growth rate are often presumed in models of intransitive competitive loops (e.g. Neumann and Jetschke 2010) and are necessary to permit co-existence of types.

Interestingly, inhibition ability ($C_t$) or resistance ($C_r$) did not trade-off with any of the life history traits measured in this study (Fig. 2a), consistent with the finding that toxin induction did not affect *in vitro* life history measures in *P. fluorescens* (Garbeva et al. 2011). Instead, we found a positive correlation between inhibitory ability ($C_t$) and overall exploitative competitiveness for *P. fluorescens*. Although unexpected, such a positive correlation is nevertheless intuitive: megacolonies invading a resident strain presumably must reach a critical size in order for any toxicity to be detectable if induction is either density dependent or if the toxic effects are concentration dependent. Cells may only reach such a critical density if their relative exploitative competitiveness enables them to do so, without which interference competitive ability is irrelevant.

#### Interactions between exploitative and interference competition.
Irrespective of the underlying mechanisms of toxicity and resistance, the frequency of these traits in a community may have large indirect effects that generate intransitive asymmetries among diverse genotypes. Several theoretical studies on intransitivity call for increased empirical research in order to test the predictions that non-hierarchical competitiveness stabilizes diversity (Laird and Schamp 2006; Laird and Schamp 2008). Our results address this by exploring some of the intransitive properties of an empirical interaction network measuring the joint outcomes of exploitative and interference competition. Instead of combining assortments of laboratory strains to explore community properties (e.g. Eisenhauer et al. 2012a; Eisenhauer et al. 2012b; Eisenhauer et al. 2013), we used a collection of natural isolates. Intransitivity in bacterial communities has been principally considered empirically with respect to bacteriocin production and resistance (Czaran et al. 2002; Kerr et al. 2002; Majeed et al. 2011).

In our interaction network, interference competition may equalize fitness differences between competitors that otherwise have asymmetric exploitative abilities (Fig. 3b). Many strains that were facilitated by an inhibitor producer to which they were resistant were also the strain whose inhibition facilitated another (Fig. 3c–d). Intransitive facilitation of the sort explored here is only possible with an intermediate frequency of toxin resistance expressed by *P. syringae* (Fig. 3d). The fact that resistance is not more common among *P. syringae* suggests a cost of resistance that did not manifest itself in the assays conducted in our study.

We show that the gains from facilitation predominantly go towards weaker resource competitors (Fig. 3c). Only in a small subset of the facilitation trios could the facilitated strain invade the producer. When the facilitated strain does not pose a competitive threat to the facilitator—as is the case most of the time here—the gains from facilitation may be short-lived. However, the overall effect of this degree of intransitive facilitation may be to prolong periods between exclusion/extinction events, elevating the diversity that is observable at any given point within the system (Laird and Schamp 2006; Laird and Schamp 2008; Laird 2014). The additional form of intransitivity found in our study is a pair of extended trios that have R–P–S invasion asymmetries, which are predicted to lead to frequency-dependent or cyclical invasion dynamics (Laird 2014). This prediction is awaiting an empirical test, and this system presents an excellent opportunity for doing so.

These conceptual implications of intransitivity speak to the large degree of diversity (both genetic and phenotypic) apparently maintained within this *Pseudomonas* spp. community, despite pairwise competitive asymmetries between many strains. Due to the seasonal nature of the phyllosphere environment in bittercress, extensive variation along an absolute fitness gradient might be expected, as spatial structure protects low fitness genotypes from global exclusion (Amarasekare 2003; Laird and Schamp 2008; Kryazhimskiy et al. 2012). Still, multiple distinct genotypes occurring within the same leaf is common (Fig. 1a), and the potential for local competitive interactions within or on leaf surfaces is large (Lindow and Brandl 2003).

The factors promoting co-existence may include competitive intransitivity mediated by exploitative and interference competition. An evolutionary consequence is that larger fitness differences may be required between genotypes or species for competitive exclusion to take place at the landscape scale (Cvijovic et al. 2018). This system is ripe for the modeling of how particular combinations of (1) life history traits and (2) inhibitor production and resistance traits can stabilize the prevalence of poor competitors embedded in non-hierarchical interaction networks. The implications of the maintenance of otherwise poor competitors for ecosystem-level traits such as productivity or trophic flow through food webs and the rate of adaptive evolution, remains a compelling topic for further study.

**Conclusions.** We found that competitive abilities of strains within a natural community of phyllosphere microbiome of *Pseudomonas* spp. varied between the two major clades present, *P. fluorescens* and *P. syringae*. Variation in competitiveness was best explained by distinct life history traits in each clade: shorter lag time in *P. fluorescens*, and increased maximum growth rate in *P. syringae*. We found no apparent life history trade-offs between growth rate and yield. The presence of different trait correlations distinct between clades illustrates the evolutionary lability of the relationships among these fundamental dimensions of competitive fitness. Conserved trait correlations within clades suggest that different life history strategies allows for clade persistence. Although speculative, the *P. fluorescens* clade may contain early colonizing strains that contest territory to a greater extent, which may serve to directly buffer against leaf colonization from potentially phytopathogenic *P. syringae*. In contrast, a high degree of inhibitor resistance among *P. syringae* may prevent local exclusion when spatial structure releases them from direct exploitative competition with *P. fluorescens*. Finally, the combination of exploitative and interference competition due to inhibitor-mediated facilitation may potentially stabilize co-existence of strains that might otherwise competitively exclude one another. Our study sheds light on the types of ecological interactions between bacterial lineages within microbiomes that should be quantified during development of microbial formations for clinical and crop enhancing purposes. 

### Methods

We measured the ecological outcome of pairwise competitive interactions among strains in this set, wherein strains competed for shared resources in spatially explicit microcosms. We quantified each strain’s ability to invade and defend against invasion and derived a composite measure of competitiveness that incorporated both invasive and defensive ability. We simultaneously measured each strain’s capacity to inhibit surrounding competitors and resist strains through inhibitory metabolites. Using independent measurements of maximum rate of increase, lag phase, and maximum yield in vitro, we then determined the underlying correlates of both exploitative and interference competitive abilities, as well as effect of genetic dissimilarity on these correlations.

**Bacterial strains.** Of the 51 *Pseudomonas* spp. strains isolated from bittercress and described by Humphrey et al. [@Humphrey14a], we selected a set of 40 (26 *P. syringae*, 14 *P. fluorescens*) that represent the relative phylogenetic diversity present in this community. The laboratory strain *P. syringae* pv. maculicola str. ES4326 (hereafter Psm4326) was used as a reference owing to its phylogenetic similarity to strains isolated from bittercress and its extensive characterization in the laboratory as a pathogen of *Arabidopsis thaliana* (Cui:2005dn, Cui:2002gp, Groen:2013bt, Groen:2015bv). All bacterial strains used in this study had undergone only one prior growth cycle after freezing following initial isolation on King's B plates from surface-sterilized homogenates of bittercress leaf samples (Humphrey et al 2014).

***In vitro* growth conditions**. We characterized canonical growth traits of each bacterial strain separately by tracking cell density through a complete lag--exponential--saturation cycle *in vitro*. We re-streaked each strain from -80°C stocks onto King's B (KB) agar plates +10 mM MgSO<sup>4</sup> and incubated at 28°C for 3 days. We picked and inoculated single colonies into 1 mL minimal media at pH 5.6 ('MM'; 10 mM fructose, 10 mM mannitol, 50 mM KPO<sub>4</sub>, 7.6 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 1.7 mM MgCl<sub>2</sub>; Mudgett:1999tu, Barrett:2011fo) and grew cultures overnight in a shaking incubator (250 rpm) at 28°C. MM at pH 5.6 induces expression of the type-III secretion system (T3SS) in a diversity of *Pseudomonas* spp. (Huynh:1989ux), in contrast to KB, which results in negligible T3SS expression. T3SS expression was important for maximizing the potential relevance of our *in vitro* assay environments to those of plants, in which T3SS expression is expected. Thus, MM was used for this and all other culture experiments in this study.

Overnight cultures of each strain in MM were spun down for 3 m at $3000 \times \text{g}$ and the supernatant replaced with 500 µL fresh MM. The density of each culture was adjusted to $\text{OD}_{600} = 0.2$ prior to 1:100 dilution into a total of 180 $\mu\text{L}$ MM inside the wells of sterile polystyrene 96-well plates (Falcon). Each 96-well plate was covered with optically clear, gas-permeable plastic tape (Sigma \#Z380059) and incubated for 60 hr in a BioTek 600 plate reader in which $\text{OD}_{600}$ measurements were taken every 5 min with continuous random orbital shaking. Identical growth assays were performed on separate days in duplicate.

**Estimating traits related to growth.** We used R package `grofit` (Kahm:2010vv) to fit smoothed functions to the bacterial growth data. Curve fits generated using logistic, Richards, Gompertz, or modified Gompertz equations failed to produce estimates with $r \geq 0.5$ and we therefore used a non-parametric locally-weighted smoothing function to estimate the following growth curve parameters: maximum growth rate $r_m$, lag phase $l$, and maximum yield $K$. Lag phase represents the length of time (min) prior to initiation of exponential growth, while maximum yield is the maximum OD<sub>600</sub> attained during 60 h of growth. Curves for long lag-phased strains never leveled off (Fig. S1, e.g. strain 17A); in these cases, $K$ was set as the final OD<sub>600</sub>. When growth trajectories exhibited multiple exponential phases (diauxic shifts), $r_m$ was estimated during the initial exponential phase (e.g. strain 20A; Fig. S1).

**Pairwise competition assays.** We conducted pairwise high-density competition assays in spatial microcosms in which a "resident" strain inoculated onto the surface of each plate competed with each "invader" strain spotted on top. In 100 mm diameter Petri dishes, we overlaid 4 mL of 0.5% (w/v) soft agar inoculated with a resident strain overtop a sterile base layer of growth medium (MM + 1% w/v agar). The top agar overlay for each plate contained a suspension of a single resident strain inoculated at $5 \times 10^{5} \text{ml}^{-1}$ while soft agar was still molten ~42°C. Once cooled, suspensions of each of the 40 invader strains were then spotted at the same concentration in 4 µL aliquots spaced every 0.5 cm in parallel rows using an 8-channel pipettor. Resident and invader strain suspensions were made from exponential phase cultures in MM 3 mL with shaking at 28°C. Plates were incubated at 28°C face up for 12 h and then face down incubation for an addition 10 days. Megacolony spots were scored by hand for growth after days 1, 3, 5, 8, and 10. Data used for the following analyses are from day 10, by which time all interactions dynamics had leveled off.

We scored growth of each invader as $0$ for no visible growth of the invader above a negative control spot containing MM alone, $0.5$ for a largely translucent 'megacolony', which reflected a definite presence of growth but which was relatively suppressed and confined to the megacolony margin, and $1$ for obvious and robust megacolony growth. Examples of each are in Fig. S2. We scored inhibition interactions as the presence of a zone of clearance (halo) $\geq 1 \text{ mm}$ surrounding the extent of the invader megacolony (Fig. S3). Inhibition interactions were ultimately scored as 0 or 1 regardless of the spatial extent of the halo, although variation in halo width was recorded. We also scored any morphological variation among megacolonies for particular strains, and later we relate such variation to the phylogenetic position of their competitors (see *Analyzing the distribution of competitive outcomes*, below).

**Calculating indexes of competitiveness.** Each strain was assayed under 40 different conditions both as resident strain and invader, comprising an interaction network with 1,600 entries (including self vs. self). One version of the interaction network represents the outcome of resource competition and details the extent of growth of each invader, while the other captures the presence or absence of inhibitory interactions indicated by zones of clearance in the resident population. For resource competitions, we calculate the invasiveness ($C_o$) and defense capacity (i.e. territoriality; $C_d$) of each strain. $C_o$ for each strain $i$ was calculated as

$$
C_{o,i} = \frac{1}{n_{ij}}\sum_{i \neq j}^{n}x_{ij}
$$

where $x_{ij} \in \{ 0,0.5,1\}$ and $n_{ij}$ is the total number of scored interactions for each strain as the invader with all non-self resident strains. $C_o$ is thus the expected value of growth attained by each strain as the invader across the population of residents.

$C_d$ quantifies the ability of each strain to resist invasion by other strains and is calculated as

$$
C_{d,j} = \frac{1}{n_{ji}}\sum_{j \neq i}^{n}(1 - x_{ji})
$$

In this equation, strain $j$ is in the resident state, and $x_{ji} \in \{ 0,0.5,1\}$ as before but with a subscript reversal, indicating the degree to which the resident prevented the growth of each invader $i$. As above, $n_{ji}$ is the number of interactions occurring between each focal resident and its non-self invaders. $C_d$ can thus be interpreted as the expected amount of growth each resident strain can prevent among the population of invaders assayed.

We then calculated an overall exploitative competition index, $C_w$, for each strain as 

$$
C_w = C_{o} - {(1 - C}_{d})
$$

where $-1 \leq C_{w} \leq 1$. These extremes represent absolute competitive inferiority ($-1$), where a strain failed to prevent any growth of any invader and similarly failed to invade any other strain, to absolute competitive dominance ($1$), where a strain fully invaded all residents and fully prevented growth of all invaders.

We also calculated $C_t$ and $C_r$ based on the interaction matrix for interference competition. Here, $C_{t}$ is the proportion of successful invasions (i.e., given growth of 0.5 or above) that also resulted in halo formation produced by invading strain and indicative of inhibition of the resident. $C_r$ for a strain is the proportion of contests with all invading inhibitor strains (i.e., all strains with $C_t > 0$) that failed to result in halo formation, which we interpreted as resistance. Anlogous to $C_w$ above, an overall interference competition index, $I_w$, was calculated for each strain as 

$$
I_{w} = C_{t} - {(1 - C}_{r})
$$

where $-1 \leq I_w \leq 1$, which is equal to the aggressiveness index ($AI$) of Vestigian et al. (2011).

**Examining trait correlations.** To examine fundamental axes of trait co-variances, we conducted principal components analysis (PCA) using the matrix of mean-centered and scaled competitive indexes and growth parameters for all strains (40 x 9 matrix) as input. We also constructed linear multiple regression models to estimate the contribution of $r_m$, $L$, and $K$ to variation among *P. syringae* and *P. fluorescens* strains in each of the overall competitive indexes $C_w$ and $I_w$.

**Analyzing the distribution of competitive outcomes**. We determined when outcomes of all pairwise interactions between strains $i$ and $j$ ($i \neq j$) took following forms: reciprocal invasibility (RI), where strains $i$ and $j$ each invade one another; reciprocal non-invasibility (RNI), where strains $i$ and $j$ cannot invade each other; and asymmetric (AS), where strain $i$ invades strain $j$ but $j$ cannot invade $i$. We constructed binomial generalized linear models (GLMs) in \texttt{R} with the canonical logit link function to estimate the probability of RI, RNI, and AS as a function of genetic distance as well as trait distance between strains. Genetic distance ($D_g$) was calculated as the pairwise uncorrected nucleotide distance between 2,690 bp of sequence comprised of four partial housekeeping gene sequences previously generated for each strain from Humphrey et al. (2014). Orthologous sequences from the genome of Psm4326 were derived from its published genome sequence (Baltrus et al 2011; RefSeq ID `NZ_AEAK00000000.1`).

Euclidean distances between each growth trait $r_m$, $L$, and $K$ for all pairs of strains were measured as $D_{ij} = \sqrt{{(x_{i} - x_{j})}^{2}}$. We first examined a binomial model for each outcome type using $D_g$ as the only fixed effect, and then computed models including each growth trait, which took the form

$$
\text{logit}\left(P(y_{\text{ij}}|x_{\text{ij}}) \right)\ \sim\ \beta_{0} + \beta_{d}x_{d} + \beta_{r_{m}}x_{r_{m}} + \beta_{L}x_{L} + \beta_{K}x_{K}
$$

To test for genetic correlations in trait values, we ran Mantel tests between pairs of trait and genetic distance matrixes in R using package `vegan`. Test statistics were compared with those generated from 1000 matrix permutations (veganCommunityEco:2012uw). We noted instances where megacolony morphology differed between strain pairings for particular isolates (e.g., *P. fluorescens* str. RM43A), and we compared the incidence of each discrete phenotype to the phylogenetic position of the competitor strains using the genetic data described above (data from Humphrey:2014ga). To test for a correlation between the phylogenetic position of competitor strains and the induced megacolony morphology of the focal strains, we conducted a permutation analysis of variance (perMANOVA) using `vegan` package with 10000 permutations.

**Inferring indirect interactions from the pairwise network.** We assembled all possible combinations of strain trios and evaluated whether their patterns of interactions fulfilled the criteria for facilitation. Specifically, we calculated the net first order linkage effects on each focal strain and in the presence of each other strain as the "competitor" strain, where the interaction between the two was mediated by a nearby third strain. Facilitation can occur by strain $A$ releasing strain $C$ from inhibition from strain $B$ (where $A$ also has to be resistant to $B$'s inhibitors), or from resource competition from superior competitor strain $B$. This analysis is agnostic to mechanism, but calculates the proportion of conditions under which facilitation of an otherwise weaker competitor is expected to arise. We also determined the prevalence of strain trios resulting in R--P--S intransitivity and synergistic inhibition, where strain $C$ is both out-competed by strain $B$ and inhibited by strain $A$, to which strain $B$ is resistant. For each strain, we calculated the net effect of antagonistic vs. facilitative indirect interactions across all possible trios and compared this to underlying fitness metrics derived from the pair-wise interaction network. Specifically, we calculated the net first order linkage effects on each focal strain and in the presence of each other strain as the "competitor" strain, where the interaction between the two was mediated by a nearby third strain. To quantify the predicted dependence of the indirect cost of susceptibility and the indirect benefit of resistance we calculated the magnitude of opportunity costs of being non-resistant. We compared these two variables to their underlying dependence on relative competitiveness in terms of resource use ($C_w$). Strong competitors lose more by being sensitive, because they would have already won most resource contests. In contrast, weaker competitors have much to gain by being resistant, but little to lose: the relative improvement in fitness increases dramatically as more contests are won owing to their increased resistance. We modeled how the costs and benefits of susceptibility and resistance depended on underlying resource competitiveness by simulating so-called first-order linkage indirect effects, where the focal interaction is impacted by a third associate strain.


### References

<div id="refs"></div>


### Acknowledgements
P.T.H. and N.K.W. gratefully acknowledge funding from the National Science Foundation (Grant Nos. DEB-1309493 to P.T.H. and DEB-1256758 to N.K.W.), the National Institute of General Medical Sciences of the National Institutes of Health (Grant No. R35GM119816 to N.K.W.), and Rocky Mountain Biological Laboratory. We are indebted to H. Briggs, K. Cromwell, A. Koning, L. Anderson, K. Niezgoda, D. Picklum and N. Alexandre; bioinformatics advice from T. K. O’Connor; and laboratory assistance from H. Pyon and A. Abidov. 

### Author Contributions
P.T.H., T.N.N. and N.K.W. designed the study. P.T.H. and T.N.N. carried out the study and analysed the data. P.T.H. and N.K.W. wrote the manuscript.

### Competing interests
The authors declare no competing interests.

<hr>

## Additional Information

### Extended data

<br>

<!-- *********************** -->
<!-- EXTENDED DATA, FIGURE 1 -->
<!-- *********************** -->

<center><img src="https://i.imgur.com/WJ4qxbP.png" width = "600"></center><br>

><strong>Extended Data Fig. 1.</strong> <strong>Pairwise competitive interactions in a phyllosphere <i>Pseudomonas</i> spp. community.</strong> Rows reflect strains in the resident state, while columns reflect strains in the invader state. Dashed red lines through interaction matrix denote within–between clade divisions for ease of visualization. Phylogeny modified from Humphrey et al. (2014).

<br>
<br>

<!-- *********************** -->
<!-- EXTENDED DATA, FIGURE 2 -->
<!-- *********************** -->

<center><img src="https://i.imgur.com/lHXFOwm.png" width = "300"></center><br>

><strong>Extended Data Fig 2.</strong> <strong>Distribution of interaction outcomes within and between <i>Pseudomonas</i> clades</strong>. Accompanying outcome counts and statistical results are displayed in Tables S1 and S2, respectively. Total strain pairings = 772 between 40 strains, with the 20 self-interactions and 8 no-data interactions removed. RI = reciprocal invasibility; RNI = reciprocal non-invasibility; Asymm. = asymmetric dominance.

<br>
<br>

<!-- *********************** -->
<!-- EXTENDED DATA, FIGURE 3 -->
<!-- *********************** -->

<center>
<img src="https://i.imgur.com/u3ywT7h.png" width = "450"></center><br>

><strong>Extended Data Fig. 3.</strong> <strong>Morphological variation observed during pairwise inhibition assays reflects strain-specific induction of motility and inhibition phenotypes in *P. fluorescens*.</strong> **a-c**. Photos of interaction plates against *P. syringae* resident strains against which *P. fluorescens* strain 43A adopted distinct megacolony morphologies (white arrows). Black arrows denote invading strains producing tox+ halos against the resident strains. **d-f**. close-ups of smooth morph (SM), wrinkly spreader-like (WS-like), and the smooth spreader (SS) morphs from above. **g**. Close-up of putative growth facilitation of non-focal strain via secretion of effectors from 43A, arising from either cell cycle modification by secreted regulators, or competitive release via killing of resident strain. H. Phylogenetic specificity of morphological induction and its relationship with toxicity for strain 43A. Phylogenetic tree from Fig. 1 (main text). Scale bars in **a–c** = 1 cm; **d–f** = 0.50 cm; **g** = 0.25 cm.

