## Manuscript number: ISMEJ-20-01631A

### Title: "Competitive hierarchies, antibiosis, and the distribution of bacterial life history traits in a microbiome"

>Dear Professor Whiteman,
>
>Thank you for submitting your manuscript to The ISME Journal, which has been reviewed by several experts in the field. We are enclosing their comments below. Although the reviewers think that the work is of potential interest, substantial concerns have also been raised that will preclude publication of the paper in its present form. We are willing to consider a revised version of the manuscript provided that you satisfy the criticisms raised by the reviewers and/or the editors. Generally, a revised manuscript should be re-submitted within 1 month. Additional time for revision may be granted to the authors upon request; please inform the editorial office as soon as possible if you anticipate the need for such an extension.
>
>Should you decide to re-submit, please include a cover letter and a point-by-point response to the referees' and editors' comments. All comments need to be addressed and if you do not agree with some of them, please specify the reasons.
>
>In addition to a clean revised manuscript, please also supply a track change PDF version of your revised manuscript so the editor and reviewers can readily assess the revisions that you have made. Please use the "Print to PDF" option for generating a PDF file in Word, rather than the "Save to PDF" option, to preserve tracked changes in the exported PDF file.
>
>Please note that The ISME Journal will only publish manuscripts if authors agree to make all data that cannot be published in the journal itself freely available in one of the public databases. Accession codes must be provided. More information about this can be found in the author instructions.
>
>The ISME Journal is committed to improving transparency in authorship. As part of our efforts in this direction, we are now requesting that all authors identified as “corresponding author” create and link their Open Researcher and Contributor Identifier (ORCID) with their account on the Manuscript Tracking System prior to acceptance. ORCID helps the scientific community achieve unambiguous attribution of all scholarly contributions. You can create and link your ORCID from the home page of the Manuscript Tracking System by clicking on “Modify my Springer Nature account” and following the instructions in the link below. Please also inform all co-authors that they can add their ORCIDs to their accounts and that they must do so prior to acceptance.
>
>https://www.springernature.com/gp/researchers/orcid/orcid-for-nature-research
>
>For more information please visit http://www.springernature.com/orcid
>
>If you experience problems in linking your ORCID, please contact the Platform Support Helpdesk.
>
>The revised manuscript should be re-submitted via our website:
>
>https://mts-isme.nature.com/cgi-bin/main.plex?el=A7Br3GzX2A3bwt2I5A9ftdyR9YWH3hvb6Z3icaWPOgZ
>
>Please feel free to contact our office if you have questions about this process.
>
>Sincerely,
>George Kowalchuk
>Senior Editor
>The ISME Journal

>Editor in Chief (JDN) comments:

>Figure 1/2 should have all bootstrap values rounded to the same % (i.e., all without decimal places for consistency). Figure 2C should have all numbers reported to same decimal places.

These changes have been made.

>Senior Editor:
>
>Although the reviewers found the results presented in your manuscript were of potential interest, they also expressed a number of important issues that would have to be addressed in revision for your manuscript to be considered further. Important issues include the clarity and presentation of the introduction and methods, as well as how results are extrapoled to more complex systems.

We thank the Senior Editor and Editor in Chief for evaluating the manuscript and for the opportunity to resubmit.

>Referee #1 (Comments to the Author):
>
>It is interesting that the authors connected different traits of bacteria with competitiveness by measuring pairwise interactions between two clades of phyllosphere pseudomonas strains. In order to quantify life history, the authors measured growth curve of each strain and calculated lag phase, growth rate and maximum yield. After that, correlations between life history and types of competitive interactions were made. This work can give implications about how to predict species interactions or community composition under competition. Moreover, this can give some information about how to construct a microbial community to defend pseudomonas invasion successfully. However, the manuscript needs further revision. Some logic was missing especially in the introduction part.
>
>The introduction part needs a thorough revision. It is only roughly written and lack of logic. For instance, the authors just wrote a tiny paragraph to demonstrate why working with plants in the beginning of introduction part. This has no connection with the 2nd paragraph. I think the importance of this study is to correlate traits with competitive interactions in the microbial community by using 40 phyllosphere bacteria. It is more necessary to develop more about competitive interactions in the beginning (why it is so important and what triats would possibly correlate with it) and introduce 40 model strains in the end of the introduction. Background about phyllosphere bacteria is also strange in line 55-60. Please rephrase this paragraph.

We thank the reviewer for this important perspective on the Introduction. We have entirely re-structured the flow of this section in light of the reviewer's suggestions.

>In line 48-49, the authors cited a reference about keystone taxa actually. I am not sure this help introduce the background.

The idea behind introducing the concept of keystone taxa was to highlight how some species generate stronger indirect effects in communities than others, and that often the magnitudes of such effects are highly skewed towards one or a small number of taxa. This idea can surely be explained more clearly, and we have strived to achieve this additional clarity in our revision.

>The authors wrote that there were various gaps highlighted in the introduction. Please make them clear and try to organize them into one or two knowledge gap or hypothesis.

We have more clearly organized the various gaps addressed be our work.

>There is no problem of incubating strains for 60h. However, the authors used the final OD600 (at 60h I think) as the maximum yield (K). It is not correct. Normally, bacteria will go into decline phase after stationary phase, which is the time that bacteria obtain the maximum yield. For instance, growth curve of strain 46A in Fig S1 had an obvious decline phase. I suggest the authors to look at the data again. In addition, the measurement on growth curve only have two replicates. I am not sure this can guarantee quality of the data. The authors did not clarify whether the two replicates (two wells) are in the same microplate or not either. Please make sure there is no pseudo-replicate. Please also make it clear how many replicates did the authors do in each experiment/measurement in the method part.

Thank you for bringing up this important point.
We are now more clear in the text (and our approach) in that we use $\text{max}[OD_{600}]$ as our measure of maximum yield, rather than the value recorded at $t = 60 h$.
Regarding replicates, we are comfortable with two replicates, as the variation among curves is remarkably small. 

>There were no panel f and g in Fig S3.

This has been fixed.

>Is there any explanation about Fig S4? It shows that it has connection with Fig 1c, but apparently there is no Fig1c in the main text. It is the same as Fig S6.

The ambiguity in figure cross-referencing has been fixed.

>The authors tried to link life history to competitiveness and defined Pseudomonas strains into two types, fast and slow life history. Can these types of traits refer to r and K strategists? It would be better that the authors conclude which type of strategist correlate with which typs of competitiveness.

We can allude more clearly to the $r$ versus $K$ strategist paradigm, which has a strong legacy in ecology but has not been applied much in the microbial sphere.
The idea breaks down in this context because, while *P. syringae* (Ps) achieves high fitness by expressing a fast growth rate (i.e., high $r$), *P. fluorescens* (Pf) competitiveness correlates with a suite of traits including the ability to inhibit neighbors via secretions.
While low $r$ is not observed among top Pf competitors, the relatively shorter lag phase among top competitors suggests that being fast (in this other dimension) is a fitness-promoting trait. 
We have attempted to introduce the idea or $r$ and $K$ selection whilst clarifying the limits of this analogy in this context, in the revised Discussion.

>There can be more explanation about the definition of competitive hierarchy and the results about rock-paper-scissors game. Please see Higgins et al., 2017 about co-occurrence of microbial species and non-transitive interactions. It would be better if the authors can provide results of three species interactions in vitro experiment to compare with the inferred data.

Unfortunately we are unable to perform additional experiments with trios *in vitro*. 
In light of this, we have undertaken a more thorough discussion of how our results relate to the many ways that hierarchies can be disrupted/modified in the context of non-transitivity (including RPS). 
We have tried to limit the scope of this discussion because the predicted influence of RPS dynamics in our system is minimal.

>Referee #2 (Comments to the Author):
>
>This study attempts to dissect competitive interactions among cooccurring bacterial strains within a leaf microbiome. The strains, although fairly closely related, seem to have distinct life history traits which makes it possible for the authors to look at correlations between traits and competition and to infer potential life history strategies. I really enjoyed reading this paper, it was a thorough dig into some often overlooked and difficult to study aspects of microbe-microbe interactions. I found some parts of the introduction and methods hard to follow (specific comments below). I found the conclusions about the potentially distinct competition strategies between the two clades fascinating, but I was less convinced by the inferred interactions between 3 strains (specific comments below).

We thank the reviewer for this feedback, and we hope to have brought some clarity in our revision.
Additionally, we have attempted to more clearly point out the limitations of our *inferred* interaction trios.
In parallel, we now point out the value of such simulations based on empirical data rather than, say, parameterizing a simulation from first principles or some other baseline.
It is our perspective that these simulations are a unique feature of our work: rather than handing off an empirical dataset to other researchers to await *in silico* study, we have included our own take.

>Major points:
>
>Spatial structure is important for several of the points here but little background is given on the spatial structuring of endophytic bacterial communities. I’m not familiar with how endophytic bacterial populations are typically structured, spatially. Is there much work on this? My understanding is that endophytic communities are often not very dense, so how frequently are populations of three different strains interacting in endophytic spaces? Particularly, how often would three strains all in one genus be interacting, just by chance? Based on the work that these isolates came from, how many Pseudomonas taxa were typically found in a given sample (a leaf or whatnot)?

This is a crucial point.
While there are still several uncertainties in the literature about these points, we will note that our work on _Cardamine cordifolia_ has shown that single leaf discs can harbor multiple distinct Ps and Pf isolates (Humphrey et al. 2014, 2020).
The observational conundrum here is, of course, that IF competitive interactions are strong, we should not expect to observe many co-infected leaf samples if our sampling scale is similar to the spatial scale of competitive exclusion.
The fact that we commonly observe multiply-infected leaf samples (but not all the time) suggests  either that competition isn't really happening; or, perhaps more likely, that we often sample leaf tissue extents that are larger than the general scale of competitive exclusion in this system.
The best answer we can give here is that distinct strains of each Ps and Pf often co-occur in single leaf samples in this system, which gives us the impression that competitive interactions among them are highly plausible even if we cannot infer their action by observational sampling alone.

>How would these interactions compare to say priority effects in shaping the community? I like the discussion of priority effects within the colonization-competition trade off, but I’m less convinced that the indirect facilitation interactions or intransitivity would actually be taking place in a leaf, based on these data. Particularly given that these interactions are being inferred from the pairwise interactions, not directly assessed. I think caveats need to be added to the discussion of these inferred interactions between 3 strains, as these conclusions seem weaker to me than the conclusions about competitive ability of the two clades or their potentially distinct life history strategies.

The *in silico* finding is naturally less compelling than the empirical one.
However, illustrating that, in principle, Pf can modify the outcome of itra-Ps interactions in potentially important ways is a fairly straight-forward and plausible scenario to highlight, as we have done with our simulation.
We have added a good deal of language to the Discussion to better contextualize the trio interaction study.

>The introduction is set up very much in terms of ecological theory and I think it could benefit from more examples or explanations related to microbiology (specifics below). I found it somewhat hard to latch onto at first, though the paper made more sense as I read more.

We have modified the introduction to more readily appeal to a microbe-focused audience, in order to facilitate the latching you refer to.

>It took me quite a while to figure out the methods for the competition assays. I think the issue was some of the terminology used. This should be clarified and it might help to have a figure in the SOM.

We have added a schematic to the Supplemental Information (Fig. S#)to illustrate the design and have clarified the language around the experimental description.

>Specific points:

>The title is a little complicated for me, it requires thinking about a lot of terminology. Since antibiosis isn’t mentioned in the abstract as a key point and doesn’t come in until the results, maybe cut it from the title? Competitive hierarchies as a function of bacterial life history traits in a microbiome?

We appreciate this suggestion and have modified the title accordingly.

>Introduction:

>The initial framing (first sentence) of the article that we can use plants to test microbiome theory fell flat for me, which may be in part from my own bias. But much of the results were discussed with specific relevance to plant microbiomes and these bacterial groups, so why not just set it up to be about plant microbiomes?

We have re-shaped the Introduction to avoid being shy about this paper being purely about plant-associated microbes.

>line 32 - I think this statement about PGPB might not make sense to a wide audience. Maybe say something about plant microbiome manipulation or probiotics already being used in agriculture?

We have made mention of PGPB-like effects in more general terms in the revision.

>lines 36-38 - “competition for shared resources and interference with another species’s ability to do so” Interference with what? Competitive ability? Or ability to access resources? It became clear to me what you meant when I read the methods overview, but I think it would be helpful to explain that you mean inhibition here.

The reference here was in general terms to interference competition, which can play out in a variety of ways. We have endeavored to make the description of interference less vague in the revision.

>lines 35-47 - I think this paragraph could benefit from a question or example that is addressed by the paper (similar to lines 56-60 below), to help orient the reader to what they are supposed to remember. As it is the paragraph reads as very broad background but I’m not sure what the paper is addressing, then later in the introduction the stated questions of the study are mostly about direct vs. indirect competition not different types of competition. I think it doesn’t help that the next paragraph is about some taxa having a large impact, rather than explicitly about inhibition, so I didn’t pick up on this theme until later.

We hope that our Introduction re-writes have alleviated this concern.

>lines 61-68 - I think this paragraph needs to be explained more or cut. You could add an example to illustrate what an intransitive loop would look like in a microbial community. I’m not sure this is a term that more microbiology focused researchers would be familiar with. The concept is explained better in the methods. However, given that you end up finding that this type of loop seems to be uncommon in the data set, and you don’t discuss it much in the discussion, maybe it would be better to leave it out of the intro?

This is a good point.
From our perspective, the intransivite loop has received the majority of attention by microbe-focused researchers as an interesting form of indirect interactions. 
We bring it up to point out that this is by far not the only indirect interaction worth understanding; far simpler motifs in communities may have a large effect (such as the simple indirect facilitation avenue studied in more detail in the paper).
That said, we have clarified the language around this topic so the reader is no longer confused about where the paper is headed.


>Methods:

>pairwise competition assays -

>I think stating that the resident was in a soft agar overlay would help the microbiology types understand what was done here without having to go into the SOM. I’m also not sure what “above a negative control spot” means, are you referring to measuring the diameter of the spot?

See comment above about our schematic SI figure.
We have also clarified the terminology used in the assay desription to be far more transparent.

>I was particularly confused by what you meant by a “megacolony.” At first I thought you meant the spots merging on the plate, but I think you mean the growth in the spot itself or beyond the spot? Why not just say spot? Or explain what you mean more, I’m not sure that megacolony is a widely used term. My preference would be to not use it, it really threw me.

We have adopted the term "spot" throughout.
Thanks for pointing this out!

>Is resident and invader the right terminology in this case, given that they were placed on the plate at about the same time? I understand what you’re going for (I think), the resident is more abundant so it theoretically has a competitive advantage against the other strain. But resident and invader bring to mind different introduction times for me, and the addition of quotes around the words doesn’t help me understand what you mean by them. Maybe something referencing relative abundance or advantage instead? Or more clarification on why this particularly experimental set up is relevant?

Like most assay setups, ours was contrived.
Our setup was not meant to resemble any particular competitive context but was made up so as to facilitate pairwise interactions at moderate scale.
We have to call the different strains something, at the end of the day; resident and invader are poor choices, admittedly, so we have adopted simpler references to the strain inoclated into the soft-agar overlay as the 'bottom' strain, and the strain spotted on top as the 'top' strain.

>I think the wording in the manuscript makes the scoring seem more subjective than it may have been. Saying “largely translucent” in line 113 suggests to me that there were less translucent spots that were scored differently than the largely translucent ones. Was there more that went into this measure than the growth being contained to the spot vs. outside the spot? How easy were these to call? Again, I think some example pictures might demonstrate how straightforward this scoring was vs. how much subjectivity went into it.

We have pointed out examples of our different spot calls (SI fig referenced in the text).
In the end it was not difficult to discretize the spot growth, since it was largely binary (with this intermediate class being far less opaque than the full spots while showing more of a "coffee stain" shape where it was spotted).
We have modified the terminology to avoid giving the impression that this process was overly subjective.

>Results:

>What’s up with strain 24A having missing data? Can you explain the somewhere?

I've looked back at this

>Figure 2: I’m having a hard time telling the difference between the no data grey and the lightest blue color. I think. 22C has no growth data?

We have made the distinction more obvious in the revised figure.

>lines 218 - Can you state what r and L are here, it’s hard to remember (although would be easier with the figure handy, it’s just not in reviewing the manuscript).

We have added clarity throughout to reduce mental load of keeping things straight.

>Discussion:

>Line 303 - saying that P. fluorescens is presumed to be a soil dwelling makes it sound like you are showing its importance on plants for the first time, but I don’t think that is accurate.

This is not our intention, but that does not change the fact that many references to Pf make more of its soil or rhizosphere association than that is shows up in plant leave.
We have clarified to ensure readers do not think we are demonstrating this for the first time.

>Referee #3 (Comments to the Author):

>Whiteman and al. performed large pairwise competition assays involving different pseudomonas strains belonging to fluorescens and syringae species. The found that P. fluorescens outcompete P. syringae in competition on agar plate. Their manuscript is entitled "Competitive hierarchies, antibiosis, and the distribution of bacterial life history traits in a microbiome" ISMEJ-20-01631A.The results represent a substantial advance in understanding the fine tuning of leaf microbiota reflected by bacterial interactions. I very much enjoyed reading this manuscript and have only a few criticisms, no more experiment are required:

We thank the reviewer for this feedback.

>General comment.

>To my opinion, the data presented are potential of interest to microbiology community. This manuscript is very well written. The ecological notions and concepts are really well transposed from to Microbiology.

>1- The title is not really enough informative and may lead to a misunderstanding of what can be found in the text as the authors did not study the leaf microbiota but competition between two pseudomonas species that are part of the latter. I suggest to write a new one that reflected the results provide inside the manuscript.

This is a fair point. We did not study these strains 'in' a microbiome, but they are 'from' a microbiome (notwithstanding the fact that similar strains probably reside in a number of different putative 'niches').
We have modified the title in light of this as well as other comments from revier #2.

>Methods :

>2- Calculation of the competitive indexes: In classical microbiology approaches, the competitive index refers to the number of bacteria recovered at “T+ incubation time” divided by the original number of bacteria at t=0. In all the indexes presented in this study, no temporality is included in the calculation and I think it is an important point because the number of bacteria can slightly change in a presence of a competitor, so based only on the colony forming visualization or a presence of an inhibition halo might not be sufficient.

We have added language to distinguish our indexes from the classical competitive index and instead highlight that these are summary statistics of the interaction patterns observed in our particular assay.

>3- Following this comment, as no starting point counting was performed, it is difficult to know if the bacteria are confronted in 1:1 or unbalanced ratios, that for sure change the fate of one of the two competitor. How the authors can argue about this?

We sought to control the starting densities as much as possible in our work to be as close to parity in starting density.
We have added text to remind readers that performing these experiments using different ratios would likely change the dynamics of the interactions.
That said, we do believe that waiting for as long as we did before scoring the outcomes will help avoid any differences arising from starting density ratios that would manifest transiently.

>4- Curiosity question, Does the growth parameter be directly included in the calculation of the competitive indexes? Could it be a way to weight/correlate the presence of an halo to the growth of the bacteria that produce inhibitory compound(s)?

I'm not sure I understand the question.
The growth parameter (measured independently, in the plate reader) does not factor in to the calculation of the competitive indexes that summarise the competitive outcomes on the plates.
If asking about whether the growth rate of Pf strains correlates with whether the strain produced inhibitory compound(s), then yes we have already included this calculation as part of Fig. 2.

>5- Several phylogenetic trees are represented in the figures, but nothing is found about how they were build? In all the figures that include trees, what are the values at each nodes of the tree? Bootstrp? How many iterations ?

The trees were built from data presented in Humphrey et al (2014), and this fact is referenced in the Methods as well as the figure caption.
We have re-summarised those methods briefly in the revised manuscript.

>Discussion:
>
>6- The discussion should be tempered a bit, as the bacterial competitions in this study was performed in a total artificial manners: on agar plate, with an amount of bacteria that is not representative of the real numbers found in plant leaf, not even the same ratio of the two competitor on plant leaves, with a diversity of bacterial strains not representative of the leaf microbiota (only Pseudomonas bacteria). I do agree that this study is important for the microbial community researcher, but here the authors provides competitions data only between bacteria that belong to a same genus.

We have added additional language to the Discussion to clarify that these experiments merely demonstrate a range of *potential* interactions among strains that naturally occur in a native pant microbiome.
We also note that pseudomonads dominate (by a very large number) the culturable and sequence-able microbiome from this and other plants.
Therefore, their ecological dominance in this sphere is a decent motivator for focusing on them specifically in this (and other) works.
Along side of this point, we have also made sure to cite other literature showing that non-pseudomonad lineages (e.g., Sphingomonas) can generate competitive effects in plant microbiomes, so that the impacts of microbial competition is by no means restricted to *Pseudomonas*.

>7- Line 303-304: The authors performed these competition assays on nutrient agar plate, the microbe interaction in a complex environment such as plant leaves is totally different as two or several bacterial strains can “avoid” competition by a different colonization pattern in time and space. I suggest to be prudent by such extrapolation and to tone it down.

Again, this study was not meant to mimic a plant environment but to demonstrate the distribution of possible interaction outcomes among strains that commonly co-occur in a native plant microbiome.
The issue of how frequently strains come into contact with one another in nature is an important one to consider.
We have added language to clarify that "the potential for the outcomes revealed here to impact microbial community patterns in leaf microbiomes depends on the timing and scale of co-colonization in actual living plants."

>8- Line 308-323: It is really hard to compare a lag time between bacteria expose to an abiotic stress like carbon source or antibiotic to a bacteria faced another one in competition. It depends of so many parameters that are not well control in this study, like the initial number of bacteria, the ratio of the competition, that are not really representative to what is happening in the phyllosphere.

Our growth experiments *in vitro* reproducibly reveal intrinsic differences in canonical features of the growth cycle among our focal strians.
As such, we expect these phenotypes, measured in this way, to relate to some features of cellular physiology expressed under natural conditions.
We do agree that, in principle, bacterial colonizing plants may express an entirely different suite of physiological patterns than in culture such that differences in lag phase in culture will not map to any commensurate trait expressed *in planta*.
This possibility aside, we make clearer that this is merely an assumption we make when considering how these interactions may play out in actual plants.

>9- Line 329-332: Very interesting hypothesis.

>Minor comments:
>- Figure 2a : Hard to distinguish grey color with the blue scale in the heat map, I suggest to change the color of the “no data” category to a more distinct color.

We have made this change.

>- Line 333: the word “exudate” is more related to what a plant secreted, I suggest to replace exudate by “metabolites”, or “antimicrobial metabolites”

We made clarified the language here.
