## Introduction revision

* Competition may well be the principal ecological force shaping microbial community structure [@Foster12a; @Coyte19a].
* But establishing support for this notion within host-associated microbiomes is hard because of the typical approaches used to investigate such communities.
* Microbiomes are typically studied indirectly (e.g., via marker gene surveys) and at taxonomic scales too broad to reveal dynamics that would arise because of competitive interactions mediated by traits.
* Further, rather than being measured directly, microbial phenotypic traits are often inferred indirectly on the basis of (e.g.) statistics summarising metabolic gene enrichment from metagenomics datasets.
* A shared goal of microbial () and general ecology () is to describe species distributions and community dynamics in terms of traits.
* Resolving this disconnect requires that we begin studying trait distributions within natural microbial communities as well as how those traits relate to competitive outcomes studied _directly_.

* One approach to address this gap between what we'd _like_ to know about microbiomes and what we _can_ know by moden molecular approaches is to revisit more traditional means of interrogating species interactions.
* For example, few studies systematically describe the level of canonical trait variation present within microbiome communities, and fewer still link such traits to the competitive interactions they mediate.
* Our view of microbial competition has been shaped by compelling demonstration of strong interactions or non-hierarchical (i.e., intransitive) dynamics within laboratory model communities.
* More rare are studies that take a direct and simpler approach to enumerating strain--strain interactions and the traits that may mediate them from a shared natural environment.


* Suite of potentially interacting bacterial strains derived from a shared leaf-associated (meta)-population and then ask how these traits mediate competitive outcomes among them.
* Using 


<hr>



- habitat filtering/species sorting
- competitive exclusion

While microbiome studies reveal strong correlational patterns among taxa, rarely do we have the opportunity to directly inspect patterns of competition among microbes isolated from a meta-community.

Inferring interactions from observational microbioime studies is notoriously fraught.

<hr>

What is this paper about?

- We see these species together a bunch
- How do they interact?
- Classic tension in ecology between abiotic and biotic forces shaping community structure.
- For microbiomes, observational studies can reveal co-occurence patterns consistent with both types of forcing.
- Even high-resolution approaches can reveal the likely action of natural selection operating within microbiomes (lineage tracing experiments, etc; Gordo, Ben Good's work, etc). 
- But competition is hard to see without actually measuring it.
- Few studies working on culturable microbiomes have done so, because these types of studies are limited.
- Even when a potential application of such knowledge would be highly useful.
- For example, direct demonstration of competitive exclusion between closely-related strains (Paharik et al. 2017) is not likely to be revealed by standard microbial diversity approaches. Direct study of competitive outcomes is necessary to establish precedent for species interactions that may be harnessed to direct 


<hr>

### Items for Discussion

_in planta_ Competition experiments within hosts will be necessary to establish whether exclusion can occur in the context of the leaf environment (e.g., Innerebner et al. 2011, https://aem.asm.org/content/77/10/3202). 


## Potential papers:

- Jeff Gore 2020
  - https://www.nature.com/articles/s41559-020-1099-4
- Otto Cordero
  - https://www.nature.com/articles/ncomms11965

